# This Makefile is for the Bio extension to perl.
#
# It was generated automatically by MakeMaker version
# 5.45 (Revision: 1.222) from the contents of
# Makefile.PL. Don't edit this file, edit Makefile.PL instead.
#
#	ANY CHANGES MADE HERE WILL BE LOST!
#
#   MakeMaker ARGV: ()
#
#   MakeMaker Parameters:

#	DISTNAME => q[bioperl]
#	NAME => q[Bio]
#	VERSION => q[0.07.1pre]
#	dist => { DIST_DEFAULT=>q[all tardist], COMPRESS=>q[gzip -9f], SUFFIX=>q[.gz] }
#	realclean => { FILES=>q[./Seq.pm.bak] }

# --- MakeMaker post_initialize section:


# --- MakeMaker const_config section:

# These definitions are from config.sh (via /usr/lib/perl5/5.6.0/i386-linux/Config.pm)

# They may have been overridden via Makefile.PL or on the command line
AR = ar
CC = gcc
CCCDLFLAGS = -fPIC
CCDLFLAGS = -rdynamic
DLEXT = so
DLSRC = dl_dlopen.xs
LD = gcc
LDDLFLAGS = -shared -L/usr/local/lib
LDFLAGS =  -L/usr/local/lib
LIBC = /lib/libc-2.2.2.so
LIB_EXT = .a
OBJ_EXT = .o
OSNAME = linux
OSVERS = 2.2.17-8smp
RANLIB = :
SO = so
EXE_EXT = 
FULL_AR = /usr/bin/ar


# --- MakeMaker constants section:
AR_STATIC_ARGS = cr
NAME = Bio
DISTNAME = bioperl
NAME_SYM = Bio
VERSION = 0.07.1pre
VERSION_SYM = 0_07_1pre
XS_VERSION = 0.07.1pre
INST_BIN = blib/bin
INST_EXE = blib/script
INST_LIB = blib/lib
INST_ARCHLIB = blib/arch
INST_SCRIPT = blib/script
PREFIX = /usr
INSTALLDIRS = site
INSTALLPRIVLIB = $(PREFIX)/lib/perl5/5.6.0
INSTALLARCHLIB = $(PREFIX)/lib/perl5/5.6.0/i386-linux
INSTALLSITELIB = $(PREFIX)/lib/perl5/site_perl/5.6.0
INSTALLSITEARCH = $(PREFIX)/lib/perl5/site_perl/5.6.0/i386-linux
INSTALLBIN = $(PREFIX)/bin
INSTALLSCRIPT = $(PREFIX)/bin
PERL_LIB = /usr/lib/perl5/5.6.0
PERL_ARCHLIB = /usr/lib/perl5/5.6.0/i386-linux
SITELIBEXP = /usr/lib/perl5/site_perl/5.6.0
SITEARCHEXP = /usr/lib/perl5/site_perl/5.6.0/i386-linux
LIBPERL_A = libperl.a
FIRST_MAKEFILE = Makefile
MAKE_APERL_FILE = Makefile.aperl
PERLMAINCC = $(CC)
PERL_INC = /usr/lib/perl5/5.6.0/i386-linux/CORE
PERL = /usr/bin/perl
FULLPERL = /usr/bin/perl
FULL_AR = /usr/bin/ar

VERSION_MACRO = VERSION
DEFINE_VERSION = -D$(VERSION_MACRO)=\"$(VERSION)\"
XS_VERSION_MACRO = XS_VERSION
XS_DEFINE_VERSION = -D$(XS_VERSION_MACRO)=\"$(XS_VERSION)\"
PERL_MALLOC_DEF = -DPERL_EXTMALLOC_DEF -Dmalloc=Perl_malloc -Dfree=Perl_mfree -Drealloc=Perl_realloc -Dcalloc=Perl_calloc

MAKEMAKER = /usr/lib/perl5/5.6.0/ExtUtils/MakeMaker.pm
MM_VERSION = 5.45

# FULLEXT = Pathname for extension directory (eg Foo/Bar/Oracle).
# BASEEXT = Basename part of FULLEXT. May be just equal FULLEXT. (eg Oracle)
# ROOTEXT = Directory part of FULLEXT with leading slash (eg /DBD)  !!! Deprecated from MM 5.32  !!!
# PARENT_NAME = NAME without BASEEXT and no trailing :: (eg Foo::Bar)
# DLBASE  = Basename part of dynamic library. May be just equal BASEEXT.
FULLEXT = Bio
BASEEXT = Bio
DLBASE = $(BASEEXT)
OBJECT = 
LDFROM = $(OBJECT)
LINKTYPE = dynamic

# Handy lists of source code files:
XS_FILES= 
C_FILES = 
O_FILES = 
H_FILES = 
HTMLLIBPODS    = 
HTMLSCRIPTPODS = 
MAN1PODS = 
MAN3PODS = Bio/AlignIO.pm \
	Bio/AlignIO/bl2seq.pm \
	Bio/AlignIO/clustalw.pm \
	Bio/AlignIO/fasta.pm \
	Bio/AlignIO/mase.pm \
	Bio/AlignIO/meme.pm \
	Bio/AlignIO/msf.pm \
	Bio/AlignIO/nexus.pm \
	Bio/AlignIO/pfam.pm \
	Bio/AlignIO/phylip.pm \
	Bio/AlignIO/prodom.pm \
	Bio/AlignIO/selex.pm \
	Bio/AlignIO/stockholm.pm \
	Bio/Annotation.pm \
	Bio/Annotation/Comment.pm \
	Bio/Annotation/DBLink.pm \
	Bio/Annotation/Reference.pm \
	Bio/DB/Ace.pm \
	Bio/DB/EMBL.pm \
	Bio/DB/Fasta.pm \
	Bio/DB/GDB.pm \
	Bio/DB/GenBank.pm \
	Bio/DB/GenPept.pm \
	Bio/DB/NCBIHelper.pm \
	Bio/DB/RandomAccessI.pm \
	Bio/DB/SeqI.pm \
	Bio/DB/SwissProt.pm \
	Bio/DB/Universal.pm \
	Bio/DB/UpdateableSeqI.pm \
	Bio/DB/WebDBSeqI.pm \
	Bio/DBLinkContainerI.pm \
	Bio/Factory/DriverFactory.pm \
	Bio/Factory/SeqAnalysisParserFactory.pm \
	Bio/Factory/SeqAnalysisParserFactoryI.pm \
	Bio/Index/Abstract.pm \
	Bio/Index/AbstractSeq.pm \
	Bio/Index/EMBL.pm \
	Bio/Index/Fasta.pm \
	Bio/Index/GenBank.pm \
	Bio/Index/SwissPfam.pm \
	Bio/Index/Swissprot.pm \
	Bio/LiveSeq/AARange.pm \
	Bio/LiveSeq/Chain.pm \
	Bio/LiveSeq/ChainI.pm \
	Bio/LiveSeq/DNA.pm \
	Bio/LiveSeq/Exon.pm \
	Bio/LiveSeq/Gene.pm \
	Bio/LiveSeq/IO/BioPerl.pm \
	Bio/LiveSeq/IO/Loader.pm \
	Bio/LiveSeq/IO/SRS.pm \
	Bio/LiveSeq/Intron.pm \
	Bio/LiveSeq/Mutation.pm \
	Bio/LiveSeq/Mutator.pm \
	Bio/LiveSeq/Prim_Transcript.pm \
	Bio/LiveSeq/Range.pm \
	Bio/LiveSeq/Repeat_Region.pm \
	Bio/LiveSeq/Repeat_Unit.pm \
	Bio/LiveSeq/SeqI.pm \
	Bio/LiveSeq/Transcript.pm \
	Bio/LiveSeq/Translation.pm \
	Bio/LocatableSeq.pm \
	Bio/Location/AvWithinCoordPolicy.pm \
	Bio/Location/CoordinatePolicyI.pm \
	Bio/Location/Fuzzy.pm \
	Bio/Location/FuzzyLocationI.pm \
	Bio/Location/NarrowestCoordPolicy.pm \
	Bio/Location/Simple.pm \
	Bio/Location/Split.pm \
	Bio/Location/SplitLocationI.pm \
	Bio/Location/WidestCoordPolicy.pm \
	Bio/LocationI.pm \
	Bio/PrimarySeq.pm \
	Bio/PrimarySeqI.pm \
	Bio/Range.pm \
	Bio/RangeI.pm \
	Bio/Root/Err.pm \
	Bio/Root/Global.pm \
	Bio/Root/IO.pm \
	Bio/Root/IOManager.pm \
	Bio/Root/Object.pm \
	Bio/Root/RootI.pm \
	Bio/Root/Utilities.pm \
	Bio/Root/Vector.pm \
	Bio/Root/Xref.pm \
	Bio/Search/Hit/Fasta.pm \
	Bio/Search/Hit/HitI.pm \
	Bio/Search/Processor.pm \
	Bio/Search/Processor/Fasta.pm \
	Bio/Search/Processor/ProcessorI.pm \
	Bio/Search/Result/Fasta.pm \
	Bio/Search/Result/ResultI.pm \
	Bio/SearchDist.pm \
	Bio/Seq.pm \
	Bio/Seq/LargePrimarySeq.pm \
	Bio/Seq/LargeSeq.pm \
	Bio/Seq/RichSeq.pm \
	Bio/Seq/RichSeqI.pm \
	Bio/SeqAnalysisParserI.pm \
	Bio/SeqFeature/Computation.pm \
	Bio/SeqFeature/Exon.pm \
	Bio/SeqFeature/FeaturePair.pm \
	Bio/SeqFeature/Gene/Exon.pm \
	Bio/SeqFeature/Gene/ExonI.pm \
	Bio/SeqFeature/Gene/GeneStructure.pm \
	Bio/SeqFeature/Gene/GeneStructureI.pm \
	Bio/SeqFeature/Gene/Transcript.pm \
	Bio/SeqFeature/Gene/TranscriptI.pm \
	Bio/SeqFeature/Generic.pm \
	Bio/SeqFeature/Intron.pm \
	Bio/SeqFeature/PolyA.pm \
	Bio/SeqFeature/Promoter.pm \
	Bio/SeqFeature/Similarity.pm \
	Bio/SeqFeature/SimilarityPair.pm \
	Bio/SeqFeature/UTR3.pm \
	Bio/SeqFeature/UTR5.pm \
	Bio/SeqFeatureI.pm \
	Bio/SeqI.pm \
	Bio/SeqIO.pm \
	Bio/SeqIO/FTHelper.pm \
	Bio/SeqIO/MultiFile.pm \
	Bio/SeqIO/ace.pm \
	Bio/SeqIO/embl.pm \
	Bio/SeqIO/fasta.pm \
	Bio/SeqIO/game.pm \
	Bio/SeqIO/game/featureHandler.pm \
	Bio/SeqIO/game/idHandler.pm \
	Bio/SeqIO/game/seqHandler.pm \
	Bio/SeqIO/gcg.pm \
	Bio/SeqIO/genbank.pm \
	Bio/SeqIO/largefasta.pm \
	Bio/SeqIO/pir.pm \
	Bio/SeqIO/raw.pm \
	Bio/SeqIO/scf.pm \
	Bio/SeqIO/swiss.pm \
	Bio/SeqUtils.pm \
	Bio/SimpleAlign.pm \
	Bio/Species.pm \
	Bio/Tools/AlignFactory.pm \
	Bio/Tools/AnalysisResult.pm \
	Bio/Tools/BPbl2seq.pm \
	Bio/Tools/BPlite.pm \
	Bio/Tools/BPlite/Iteration.pm \
	Bio/Tools/BPpsilite.pm \
	Bio/Tools/Blast.pm \
	Bio/Tools/Blast/HSP.pm \
	Bio/Tools/Blast/HTML.pm \
	Bio/Tools/Blast/Run/LocalBlast.pm \
	Bio/Tools/Blast/Run/Webblast.pm \
	Bio/Tools/Blast/Sbjct.pm \
	Bio/Tools/CodonTable.pm \
	Bio/Tools/ESTScan.pm \
	Bio/Tools/Fasta.pm \
	Bio/Tools/GFF.pm \
	Bio/Tools/Genemark.pm \
	Bio/Tools/Genscan.pm \
	Bio/Tools/Grail.pm \
	Bio/Tools/HMMER/Domain.pm \
	Bio/Tools/HMMER/Results.pm \
	Bio/Tools/HMMER/Set.pm \
	Bio/Tools/IUPAC.pm \
	Bio/Tools/MZEF.pm \
	Bio/Tools/OddCodes.pm \
	Bio/Tools/PPSEARCH/Parse.pm \
	Bio/Tools/PRFSCAN/Parse.pm \
	Bio/Tools/PRINTS/Parse.pm \
	Bio/Tools/Prediction/Exon.pm \
	Bio/Tools/Prediction/Gene.pm \
	Bio/Tools/RestrictionEnzyme.pm \
	Bio/Tools/Run/Alignment/Clustalw.pm \
	Bio/Tools/Run/Alignment/TCoffee.pm \
	Bio/Tools/Run/RemoteBlast.pm \
	Bio/Tools/Run/StandAloneBlast.pm \
	Bio/Tools/SeqAnal.pm \
	Bio/Tools/SeqPattern.pm \
	Bio/Tools/SeqStats.pm \
	Bio/Tools/SeqWords.pm \
	Bio/Tools/Sigcleave.pm \
	Bio/Tools/Sim4/Exon.pm \
	Bio/Tools/Sim4/Results.pm \
	Bio/Tools/WWW.pm \
	Bio/Tools/pSW.pm \
	Bio/UnivAln.pm \
	Bio/Variation/AAChange.pm \
	Bio/Variation/AAReverseMutate.pm \
	Bio/Variation/Allele.pm \
	Bio/Variation/DNAMutation.pm \
	Bio/Variation/IO.pm \
	Bio/Variation/IO/flat.pm \
	Bio/Variation/IO/xml.pm \
	Bio/Variation/RNAChange.pm \
	Bio/Variation/SeqDiff.pm \
	Bio/Variation/VariantI.pm \
	bioback.pod \
	biodesign.pod \
	bioperl.pod \
	biostart.pod \
	bptutorial.pl
HTMLEXT = html
INST_MAN1DIR = blib/man1
INSTALLMAN1DIR = $(PREFIX)/share/man/man1
MAN1EXT = 1
INST_MAN3DIR = blib/man3
INSTALLMAN3DIR = $(PREFIX)/share/man/man3
MAN3EXT = 3pm
PERM_RW = 644
PERM_RWX = 755

# work around a famous dec-osf make(1) feature(?):
makemakerdflt: all

.SUFFIXES: .xs .c .C .cpp .cxx .cc $(OBJ_EXT)

# Nick wanted to get rid of .PRECIOUS. I don't remember why. I seem to recall, that
# some make implementations will delete the Makefile when we rebuild it. Because
# we call false(1) when we rebuild it. So make(1) is not completely wrong when it
# does so. Our milage may vary.
# .PRECIOUS: Makefile    # seems to be not necessary anymore

.PHONY: all config static dynamic test linkext manifest

# Where is the Config information that we are using/depend on
CONFIGDEP = $(PERL_ARCHLIB)/Config.pm $(PERL_INC)/config.h

# Where to put things:
INST_LIBDIR      = $(INST_LIB)
INST_ARCHLIBDIR  = $(INST_ARCHLIB)

INST_AUTODIR     = $(INST_LIB)/auto/$(FULLEXT)
INST_ARCHAUTODIR = $(INST_ARCHLIB)/auto/$(FULLEXT)

INST_STATIC  =
INST_DYNAMIC =
INST_BOOT    =

EXPORT_LIST = 

PERL_ARCHIVE = 

TO_INST_PM = Bio/AlignIO.pm \
	Bio/AlignIO/bl2seq.pm \
	Bio/AlignIO/clustalw.pm \
	Bio/AlignIO/fasta.pm \
	Bio/AlignIO/mase.pm \
	Bio/AlignIO/meme.pm \
	Bio/AlignIO/msf.pm \
	Bio/AlignIO/nexus.pm \
	Bio/AlignIO/pfam.pm \
	Bio/AlignIO/phylip.pm \
	Bio/AlignIO/prodom.pm \
	Bio/AlignIO/selex.pm \
	Bio/AlignIO/stockholm.pm \
	Bio/Annotation.pm \
	Bio/Annotation/Comment.pm \
	Bio/Annotation/DBLink.pm \
	Bio/Annotation/Reference.pm \
	Bio/DB/Ace.pm \
	Bio/DB/EMBL.pm \
	Bio/DB/Fasta.pm \
	Bio/DB/GDB.pm \
	Bio/DB/GenBank.pm \
	Bio/DB/GenPept.pm \
	Bio/DB/NCBIHelper.pm \
	Bio/DB/RandomAccessI.pm \
	Bio/DB/SeqI.pm \
	Bio/DB/SwissProt.pm \
	Bio/DB/Universal.pm \
	Bio/DB/UpdateableSeqI.pm \
	Bio/DB/WebDBSeqI.pm \
	Bio/DBLinkContainerI.pm \
	Bio/Factory/DriverFactory.pm \
	Bio/Factory/SeqAnalysisParserFactory.pm \
	Bio/Factory/SeqAnalysisParserFactoryI.pm \
	Bio/Index/Abstract.pm \
	Bio/Index/AbstractSeq.pm \
	Bio/Index/EMBL.pm \
	Bio/Index/Fasta.pm \
	Bio/Index/GenBank.pm \
	Bio/Index/SwissPfam.pm \
	Bio/Index/Swissprot.pm \
	Bio/LiveSeq/AARange.pm \
	Bio/LiveSeq/Chain.pm \
	Bio/LiveSeq/ChainI.pm \
	Bio/LiveSeq/DNA.pm \
	Bio/LiveSeq/Exon.pm \
	Bio/LiveSeq/Gene.pm \
	Bio/LiveSeq/IO/BioPerl.pm \
	Bio/LiveSeq/IO/Loader.pm \
	Bio/LiveSeq/IO/SRS.pm \
	Bio/LiveSeq/Intron.pm \
	Bio/LiveSeq/Mutation.pm \
	Bio/LiveSeq/Mutator.pm \
	Bio/LiveSeq/Prim_Transcript.pm \
	Bio/LiveSeq/Range.pm \
	Bio/LiveSeq/Repeat_Region.pm \
	Bio/LiveSeq/Repeat_Unit.pm \
	Bio/LiveSeq/SeqI.pm \
	Bio/LiveSeq/Transcript.pm \
	Bio/LiveSeq/Translation.pm \
	Bio/LocatableSeq.pm \
	Bio/Location/AvWithinCoordPolicy.pm \
	Bio/Location/CoordinatePolicyI.pm \
	Bio/Location/Fuzzy.pm \
	Bio/Location/FuzzyLocationI.pm \
	Bio/Location/NarrowestCoordPolicy.pm \
	Bio/Location/Simple.pm \
	Bio/Location/Split.pm \
	Bio/Location/SplitLocationI.pm \
	Bio/Location/WidestCoordPolicy.pm \
	Bio/LocationI.pm \
	Bio/PrimarySeq.pm \
	Bio/PrimarySeqI.pm \
	Bio/Range.pm \
	Bio/RangeI.pm \
	Bio/Root/Err.pm \
	Bio/Root/Global.pm \
	Bio/Root/IO.pm \
	Bio/Root/IOManager.pm \
	Bio/Root/Object.pm \
	Bio/Root/RootI.pm \
	Bio/Root/Utilities.pm \
	Bio/Root/Vector.pm \
	Bio/Root/Xref.pm \
	Bio/Search/Hit/Fasta.pm \
	Bio/Search/Hit/HitI.pm \
	Bio/Search/Processor.pm \
	Bio/Search/Processor/Fasta.pm \
	Bio/Search/Processor/ProcessorI.pm \
	Bio/Search/Result/Fasta.pm \
	Bio/Search/Result/ResultI.pm \
	Bio/SearchDist.pm \
	Bio/Seq.pm \
	Bio/Seq/LargePrimarySeq.pm \
	Bio/Seq/LargeSeq.pm \
	Bio/Seq/RichSeq.pm \
	Bio/Seq/RichSeqI.pm \
	Bio/SeqAnalysisParserI.pm \
	Bio/SeqFeature/Computation.pm \
	Bio/SeqFeature/Exon.pm \
	Bio/SeqFeature/FeaturePair.pm \
	Bio/SeqFeature/Gene/Exon.pm \
	Bio/SeqFeature/Gene/ExonI.pm \
	Bio/SeqFeature/Gene/GeneStructure.pm \
	Bio/SeqFeature/Gene/GeneStructureI.pm \
	Bio/SeqFeature/Gene/Transcript.pm \
	Bio/SeqFeature/Gene/TranscriptI.pm \
	Bio/SeqFeature/Generic.pm \
	Bio/SeqFeature/Intron.pm \
	Bio/SeqFeature/PolyA.pm \
	Bio/SeqFeature/Promoter.pm \
	Bio/SeqFeature/Similarity.pm \
	Bio/SeqFeature/SimilarityPair.pm \
	Bio/SeqFeature/UTR3.pm \
	Bio/SeqFeature/UTR5.pm \
	Bio/SeqFeatureI.pm \
	Bio/SeqI.pm \
	Bio/SeqIO.pm \
	Bio/SeqIO/FTHelper.pm \
	Bio/SeqIO/MultiFile.pm \
	Bio/SeqIO/ace.pm \
	Bio/SeqIO/embl.pm \
	Bio/SeqIO/fasta.pm \
	Bio/SeqIO/game.pm \
	Bio/SeqIO/game/featureHandler.pm \
	Bio/SeqIO/game/idHandler.pm \
	Bio/SeqIO/game/seqHandler.pm \
	Bio/SeqIO/gcg.pm \
	Bio/SeqIO/genbank.pm \
	Bio/SeqIO/largefasta.pm \
	Bio/SeqIO/pir.pm \
	Bio/SeqIO/raw.pm \
	Bio/SeqIO/scf.pm \
	Bio/SeqIO/swiss.pm \
	Bio/SeqUtils.pm \
	Bio/SimpleAlign.pm \
	Bio/Species.pm \
	Bio/Tools/AlignFactory.pm \
	Bio/Tools/AnalysisResult.pm \
	Bio/Tools/BPbl2seq.pm \
	Bio/Tools/BPlite.pm \
	Bio/Tools/BPlite/HSP.pm \
	Bio/Tools/BPlite/Iteration.pm \
	Bio/Tools/BPlite/Sbjct.pm \
	Bio/Tools/BPpsilite.pm \
	Bio/Tools/Blast.pm \
	Bio/Tools/Blast/CHANGES \
	Bio/Tools/Blast/HSP.pm \
	Bio/Tools/Blast/HTML.pm \
	Bio/Tools/Blast/README \
	Bio/Tools/Blast/Run/LocalBlast.pm \
	Bio/Tools/Blast/Run/Makefile.PL \
	Bio/Tools/Blast/Run/Webblast.pm \
	Bio/Tools/Blast/Run/postclient.pl \
	Bio/Tools/Blast/Sbjct.pm \
	Bio/Tools/CodonTable.pm \
	Bio/Tools/ESTScan.pm \
	Bio/Tools/Fasta.pm \
	Bio/Tools/GFF.pm \
	Bio/Tools/Genemark.pm \
	Bio/Tools/Genscan.pm \
	Bio/Tools/Grail.pm \
	Bio/Tools/HMMER/Domain.pm \
	Bio/Tools/HMMER/Results.pm \
	Bio/Tools/HMMER/Set.pm \
	Bio/Tools/IUPAC.pm \
	Bio/Tools/MZEF.pm \
	Bio/Tools/OddCodes.pm \
	Bio/Tools/PPSEARCH/Parse.pm \
	Bio/Tools/PRFSCAN/Parse.pm \
	Bio/Tools/PRINTS/Parse.pm \
	Bio/Tools/Prediction/Exon.pm \
	Bio/Tools/Prediction/Gene.pm \
	Bio/Tools/RestrictionEnzyme.pm \
	Bio/Tools/Run/Alignment/Clustalw.pm \
	Bio/Tools/Run/Alignment/TCoffee.pm \
	Bio/Tools/Run/RemoteBlast.pm \
	Bio/Tools/Run/StandAloneBlast.pm \
	Bio/Tools/SeqAnal.pm \
	Bio/Tools/SeqPattern.pm \
	Bio/Tools/SeqStats.pm \
	Bio/Tools/SeqWords.pm \
	Bio/Tools/Sigcleave.pm \
	Bio/Tools/Sim4/Exon.pm \
	Bio/Tools/Sim4/Results.pm \
	Bio/Tools/WWW.pm \
	Bio/Tools/pSW.pm \
	Bio/UnivAln.pm \
	Bio/Variation/AAChange.pm \
	Bio/Variation/AAReverseMutate.pm \
	Bio/Variation/Allele.pm \
	Bio/Variation/DNAMutation.pm \
	Bio/Variation/IO.pm \
	Bio/Variation/IO/flat.pm \
	Bio/Variation/IO/xml.pm \
	Bio/Variation/README \
	Bio/Variation/RNAChange.pm \
	Bio/Variation/SeqDiff.pm \
	Bio/Variation/VariantI.pm \
	bioback.pod \
	biodesign.pod \
	bioperl.pod \
	biostart.pod \
	bptutorial.pl

PM_TO_BLIB = Bio/Annotation/Reference.pm \
	$(INST_LIBDIR)/Bio/Annotation/Reference.pm \
	Bio/Root/Err.pm \
	$(INST_LIBDIR)/Bio/Root/Err.pm \
	Bio/Tools/Genscan.pm \
	$(INST_LIBDIR)/Bio/Tools/Genscan.pm \
	bioback.pod \
	$(INST_LIBDIR)/bioback.pod \
	Bio/AlignIO/meme.pm \
	$(INST_LIBDIR)/Bio/AlignIO/meme.pm \
	Bio/AlignIO/fasta.pm \
	$(INST_LIBDIR)/Bio/AlignIO/fasta.pm \
	Bio/Location/NarrowestCoordPolicy.pm \
	$(INST_LIBDIR)/Bio/Location/NarrowestCoordPolicy.pm \
	Bio/AlignIO/clustalw.pm \
	$(INST_LIBDIR)/Bio/AlignIO/clustalw.pm \
	Bio/Tools/Blast/Run/postclient.pl \
	$(INST_LIBDIR)/Bio/Tools/Blast/Run/postclient.pl \
	Bio/Annotation/DBLink.pm \
	$(INST_LIBDIR)/Bio/Annotation/DBLink.pm \
	Bio/DB/Fasta.pm \
	$(INST_LIBDIR)/Bio/DB/Fasta.pm \
	Bio/LiveSeq/Intron.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Intron.pm \
	Bio/AlignIO/mase.pm \
	$(INST_LIBDIR)/Bio/AlignIO/mase.pm \
	Bio/LiveSeq/Gene.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Gene.pm \
	Bio/Tools/PPSEARCH/Parse.pm \
	$(INST_LIBDIR)/Bio/Tools/PPSEARCH/Parse.pm \
	Bio/Tools/HMMER/Domain.pm \
	$(INST_LIBDIR)/Bio/Tools/HMMER/Domain.pm \
	Bio/Variation/IO.pm \
	$(INST_LIBDIR)/Bio/Variation/IO.pm \
	Bio/SeqFeature/Exon.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Exon.pm \
	Bio/DB/GDB.pm \
	$(INST_LIBDIR)/Bio/DB/GDB.pm \
	Bio/Tools/Blast/Run/LocalBlast.pm \
	$(INST_LIBDIR)/Bio/Tools/Blast/Run/LocalBlast.pm \
	Bio/Variation/AAChange.pm \
	$(INST_LIBDIR)/Bio/Variation/AAChange.pm \
	Bio/SeqFeature/Generic.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Generic.pm \
	Bio/SeqFeature/Gene/GeneStructure.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Gene/GeneStructure.pm \
	Bio/Tools/Blast/Sbjct.pm \
	$(INST_LIBDIR)/Bio/Tools/Blast/Sbjct.pm \
	Bio/Index/EMBL.pm \
	$(INST_LIBDIR)/Bio/Index/EMBL.pm \
	Bio/Tools/Run/RemoteBlast.pm \
	$(INST_LIBDIR)/Bio/Tools/Run/RemoteBlast.pm \
	Bio/SeqI.pm \
	$(INST_LIBDIR)/Bio/SeqI.pm \
	Bio/SeqFeature/PolyA.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/PolyA.pm \
	Bio/Tools/OddCodes.pm \
	$(INST_LIBDIR)/Bio/Tools/OddCodes.pm \
	biostart.pod \
	$(INST_LIBDIR)/biostart.pod \
	Bio/Root/Xref.pm \
	$(INST_LIBDIR)/Bio/Root/Xref.pm \
	Bio/Root/Global.pm \
	$(INST_LIBDIR)/Bio/Root/Global.pm \
	Bio/Seq.pm \
	$(INST_LIBDIR)/Bio/Seq.pm \
	Bio/SeqIO/gcg.pm \
	$(INST_LIBDIR)/Bio/SeqIO/gcg.pm \
	Bio/Tools/Blast/Run/Webblast.pm \
	$(INST_LIBDIR)/Bio/Tools/Blast/Run/Webblast.pm \
	Bio/Root/Vector.pm \
	$(INST_LIBDIR)/Bio/Root/Vector.pm \
	Bio/Tools/HMMER/Set.pm \
	$(INST_LIBDIR)/Bio/Tools/HMMER/Set.pm \
	Bio/SeqFeature/UTR3.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/UTR3.pm \
	Bio/SeqIO/scf.pm \
	$(INST_LIBDIR)/Bio/SeqIO/scf.pm \
	Bio/SeqIO/FTHelper.pm \
	$(INST_LIBDIR)/Bio/SeqIO/FTHelper.pm \
	Bio/RangeI.pm \
	$(INST_LIBDIR)/Bio/RangeI.pm \
	Bio/LiveSeq/IO/SRS.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/IO/SRS.pm \
	Bio/LiveSeq/Translation.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Translation.pm \
	Bio/LiveSeq/Chain.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Chain.pm \
	Bio/PrimarySeq.pm \
	$(INST_LIBDIR)/Bio/PrimarySeq.pm \
	Bio/Tools/BPlite/HSP.pm \
	$(INST_LIBDIR)/Bio/Tools/BPlite/HSP.pm \
	Bio/Tools/Sigcleave.pm \
	$(INST_LIBDIR)/Bio/Tools/Sigcleave.pm \
	Bio/Root/Object.pm \
	$(INST_LIBDIR)/Bio/Root/Object.pm \
	Bio/LiveSeq/Mutation.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Mutation.pm \
	Bio/Location/AvWithinCoordPolicy.pm \
	$(INST_LIBDIR)/Bio/Location/AvWithinCoordPolicy.pm \
	Bio/SeqFeature/Promoter.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Promoter.pm \
	Bio/Tools/IUPAC.pm \
	$(INST_LIBDIR)/Bio/Tools/IUPAC.pm \
	Bio/SeqIO/ace.pm \
	$(INST_LIBDIR)/Bio/SeqIO/ace.pm \
	Bio/LiveSeq/SeqI.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/SeqI.pm \
	Bio/Location/FuzzyLocationI.pm \
	$(INST_LIBDIR)/Bio/Location/FuzzyLocationI.pm \
	Bio/Tools/Blast/Run/Makefile.PL \
	$(INST_LIBDIR)/Bio/Tools/Blast/Run/Makefile.PL \
	Bio/Tools/Genemark.pm \
	$(INST_LIBDIR)/Bio/Tools/Genemark.pm \
	Bio/Tools/SeqPattern.pm \
	$(INST_LIBDIR)/Bio/Tools/SeqPattern.pm \
	Bio/SeqFeature/SimilarityPair.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/SimilarityPair.pm \
	Bio/SeqIO/pir.pm \
	$(INST_LIBDIR)/Bio/SeqIO/pir.pm \
	Bio/Tools/AnalysisResult.pm \
	$(INST_LIBDIR)/Bio/Tools/AnalysisResult.pm \
	Bio/Tools/SeqAnal.pm \
	$(INST_LIBDIR)/Bio/Tools/SeqAnal.pm \
	Bio/SeqFeature/Gene/Transcript.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Gene/Transcript.pm \
	Bio/Search/Processor/ProcessorI.pm \
	$(INST_LIBDIR)/Bio/Search/Processor/ProcessorI.pm \
	Bio/Seq/RichSeqI.pm \
	$(INST_LIBDIR)/Bio/Seq/RichSeqI.pm \
	Bio/Location/Simple.pm \
	$(INST_LIBDIR)/Bio/Location/Simple.pm \
	Bio/LiveSeq/IO/Loader.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/IO/Loader.pm \
	Bio/Tools/WWW.pm \
	$(INST_LIBDIR)/Bio/Tools/WWW.pm \
	Bio/Variation/IO/flat.pm \
	$(INST_LIBDIR)/Bio/Variation/IO/flat.pm \
	Bio/LiveSeq/AARange.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/AARange.pm \
	Bio/Index/AbstractSeq.pm \
	$(INST_LIBDIR)/Bio/Index/AbstractSeq.pm \
	Bio/SeqAnalysisParserI.pm \
	$(INST_LIBDIR)/Bio/SeqAnalysisParserI.pm \
	Bio/Tools/GFF.pm \
	$(INST_LIBDIR)/Bio/Tools/GFF.pm \
	Bio/Seq/RichSeq.pm \
	$(INST_LIBDIR)/Bio/Seq/RichSeq.pm \
	Bio/Tools/ESTScan.pm \
	$(INST_LIBDIR)/Bio/Tools/ESTScan.pm \
	Bio/Root/RootI.pm \
	$(INST_LIBDIR)/Bio/Root/RootI.pm \
	Bio/Variation/AAReverseMutate.pm \
	$(INST_LIBDIR)/Bio/Variation/AAReverseMutate.pm \
	Bio/Variation/IO/xml.pm \
	$(INST_LIBDIR)/Bio/Variation/IO/xml.pm \
	Bio/DB/EMBL.pm \
	$(INST_LIBDIR)/Bio/DB/EMBL.pm \
	Bio/Tools/BPpsilite.pm \
	$(INST_LIBDIR)/Bio/Tools/BPpsilite.pm \
	Bio/Tools/SeqWords.pm \
	$(INST_LIBDIR)/Bio/Tools/SeqWords.pm \
	Bio/DB/UpdateableSeqI.pm \
	$(INST_LIBDIR)/Bio/DB/UpdateableSeqI.pm \
	Bio/SeqFeature/Computation.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Computation.pm \
	Bio/DB/GenBank.pm \
	$(INST_LIBDIR)/Bio/DB/GenBank.pm \
	Bio/LiveSeq/Prim_Transcript.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Prim_Transcript.pm \
	Bio/SeqUtils.pm \
	$(INST_LIBDIR)/Bio/SeqUtils.pm \
	Bio/Variation/VariantI.pm \
	$(INST_LIBDIR)/Bio/Variation/VariantI.pm \
	Bio/LiveSeq/Transcript.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Transcript.pm \
	Bio/LocatableSeq.pm \
	$(INST_LIBDIR)/Bio/LocatableSeq.pm \
	Bio/Tools/Sim4/Exon.pm \
	$(INST_LIBDIR)/Bio/Tools/Sim4/Exon.pm \
	Bio/Tools/MZEF.pm \
	$(INST_LIBDIR)/Bio/Tools/MZEF.pm \
	Bio/Tools/CodonTable.pm \
	$(INST_LIBDIR)/Bio/Tools/CodonTable.pm \
	Bio/Tools/Run/StandAloneBlast.pm \
	$(INST_LIBDIR)/Bio/Tools/Run/StandAloneBlast.pm \
	Bio/Variation/README \
	$(INST_LIBDIR)/Bio/Variation/README \
	Bio/AlignIO/nexus.pm \
	$(INST_LIBDIR)/Bio/AlignIO/nexus.pm \
	Bio/Annotation/Comment.pm \
	$(INST_LIBDIR)/Bio/Annotation/Comment.pm \
	Bio/Factory/DriverFactory.pm \
	$(INST_LIBDIR)/Bio/Factory/DriverFactory.pm \
	Bio/Tools/PRINTS/Parse.pm \
	$(INST_LIBDIR)/Bio/Tools/PRINTS/Parse.pm \
	Bio/DB/SeqI.pm \
	$(INST_LIBDIR)/Bio/DB/SeqI.pm \
	Bio/AlignIO/prodom.pm \
	$(INST_LIBDIR)/Bio/AlignIO/prodom.pm \
	Bio/Location/WidestCoordPolicy.pm \
	$(INST_LIBDIR)/Bio/Location/WidestCoordPolicy.pm \
	Bio/Root/IOManager.pm \
	$(INST_LIBDIR)/Bio/Root/IOManager.pm \
	Bio/LiveSeq/Repeat_Region.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Repeat_Region.pm \
	Bio/Factory/SeqAnalysisParserFactoryI.pm \
	$(INST_LIBDIR)/Bio/Factory/SeqAnalysisParserFactoryI.pm \
	Bio/SeqIO/game/seqHandler.pm \
	$(INST_LIBDIR)/Bio/SeqIO/game/seqHandler.pm \
	Bio/LocationI.pm \
	$(INST_LIBDIR)/Bio/LocationI.pm \
	Bio/Tools/Sim4/Results.pm \
	$(INST_LIBDIR)/Bio/Tools/Sim4/Results.pm \
	Bio/DB/Universal.pm \
	$(INST_LIBDIR)/Bio/DB/Universal.pm \
	Bio/Tools/Prediction/Exon.pm \
	$(INST_LIBDIR)/Bio/Tools/Prediction/Exon.pm \
	Bio/SeqIO/game/idHandler.pm \
	$(INST_LIBDIR)/Bio/SeqIO/game/idHandler.pm \
	Bio/Search/Result/Fasta.pm \
	$(INST_LIBDIR)/Bio/Search/Result/Fasta.pm \
	Bio/AlignIO/phylip.pm \
	$(INST_LIBDIR)/Bio/AlignIO/phylip.pm \
	Bio/Tools/Blast/HSP.pm \
	$(INST_LIBDIR)/Bio/Tools/Blast/HSP.pm \
	Bio/DB/WebDBSeqI.pm \
	$(INST_LIBDIR)/Bio/DB/WebDBSeqI.pm \
	Bio/SeqFeatureI.pm \
	$(INST_LIBDIR)/Bio/SeqFeatureI.pm \
	Bio/SeqIO/swiss.pm \
	$(INST_LIBDIR)/Bio/SeqIO/swiss.pm \
	Bio/Search/Hit/Fasta.pm \
	$(INST_LIBDIR)/Bio/Search/Hit/Fasta.pm \
	Bio/Location/Split.pm \
	$(INST_LIBDIR)/Bio/Location/Split.pm \
	Bio/Tools/Grail.pm \
	$(INST_LIBDIR)/Bio/Tools/Grail.pm \
	Bio/DB/GenPept.pm \
	$(INST_LIBDIR)/Bio/DB/GenPept.pm \
	Bio/Tools/SeqStats.pm \
	$(INST_LIBDIR)/Bio/Tools/SeqStats.pm \
	Bio/LiveSeq/ChainI.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/ChainI.pm \
	Bio/Variation/SeqDiff.pm \
	$(INST_LIBDIR)/Bio/Variation/SeqDiff.pm \
	Bio/PrimarySeqI.pm \
	$(INST_LIBDIR)/Bio/PrimarySeqI.pm \
	Bio/Variation/DNAMutation.pm \
	$(INST_LIBDIR)/Bio/Variation/DNAMutation.pm \
	Bio/Search/Result/ResultI.pm \
	$(INST_LIBDIR)/Bio/Search/Result/ResultI.pm \
	Bio/Search/Processor.pm \
	$(INST_LIBDIR)/Bio/Search/Processor.pm \
	Bio/Tools/Prediction/Gene.pm \
	$(INST_LIBDIR)/Bio/Tools/Prediction/Gene.pm \
	Bio/UnivAln.pm \
	$(INST_LIBDIR)/Bio/UnivAln.pm \
	Bio/Variation/Allele.pm \
	$(INST_LIBDIR)/Bio/Variation/Allele.pm \
	Bio/SeqFeature/FeaturePair.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/FeaturePair.pm \
	biodesign.pod \
	$(INST_LIBDIR)/biodesign.pod \
	Bio/AlignIO/msf.pm \
	$(INST_LIBDIR)/Bio/AlignIO/msf.pm \
	Bio/SeqIO/embl.pm \
	$(INST_LIBDIR)/Bio/SeqIO/embl.pm \
	Bio/LiveSeq/Repeat_Unit.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Repeat_Unit.pm \
	Bio/Tools/BPlite/Sbjct.pm \
	$(INST_LIBDIR)/Bio/Tools/BPlite/Sbjct.pm \
	Bio/Tools/Fasta.pm \
	$(INST_LIBDIR)/Bio/Tools/Fasta.pm \
	Bio/SeqFeature/Gene/Exon.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Gene/Exon.pm \
	Bio/Root/IO.pm \
	$(INST_LIBDIR)/Bio/Root/IO.pm \
	Bio/Search/Hit/HitI.pm \
	$(INST_LIBDIR)/Bio/Search/Hit/HitI.pm \
	Bio/DB/NCBIHelper.pm \
	$(INST_LIBDIR)/Bio/DB/NCBIHelper.pm \
	Bio/Tools/Run/Alignment/TCoffee.pm \
	$(INST_LIBDIR)/Bio/Tools/Run/Alignment/TCoffee.pm \
	Bio/SeqIO/largefasta.pm \
	$(INST_LIBDIR)/Bio/SeqIO/largefasta.pm \
	Bio/Index/Swissprot.pm \
	$(INST_LIBDIR)/Bio/Index/Swissprot.pm \
	Bio/Tools/Run/Alignment/Clustalw.pm \
	$(INST_LIBDIR)/Bio/Tools/Run/Alignment/Clustalw.pm \
	Bio/Species.pm \
	$(INST_LIBDIR)/Bio/Species.pm \
	Bio/Search/Processor/Fasta.pm \
	$(INST_LIBDIR)/Bio/Search/Processor/Fasta.pm \
	Bio/Index/Abstract.pm \
	$(INST_LIBDIR)/Bio/Index/Abstract.pm \
	Bio/Seq/LargePrimarySeq.pm \
	$(INST_LIBDIR)/Bio/Seq/LargePrimarySeq.pm \
	Bio/LiveSeq/Mutator.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Mutator.pm \
	Bio/AlignIO.pm \
	$(INST_LIBDIR)/Bio/AlignIO.pm \
	Bio/Tools/pSW.pm \
	$(INST_LIBDIR)/Bio/Tools/pSW.pm \
	Bio/Tools/BPlite/Iteration.pm \
	$(INST_LIBDIR)/Bio/Tools/BPlite/Iteration.pm \
	Bio/Factory/SeqAnalysisParserFactory.pm \
	$(INST_LIBDIR)/Bio/Factory/SeqAnalysisParserFactory.pm \
	Bio/SimpleAlign.pm \
	$(INST_LIBDIR)/Bio/SimpleAlign.pm \
	Bio/SeqFeature/UTR5.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/UTR5.pm \
	Bio/Index/GenBank.pm \
	$(INST_LIBDIR)/Bio/Index/GenBank.pm \
	Bio/AlignIO/stockholm.pm \
	$(INST_LIBDIR)/Bio/AlignIO/stockholm.pm \
	Bio/SeqIO.pm \
	$(INST_LIBDIR)/Bio/SeqIO.pm \
	Bio/Index/Fasta.pm \
	$(INST_LIBDIR)/Bio/Index/Fasta.pm \
	Bio/SearchDist.pm \
	$(INST_LIBDIR)/Bio/SearchDist.pm \
	Bio/LiveSeq/IO/BioPerl.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/IO/BioPerl.pm \
	bptutorial.pl \
	$(INST_LIBDIR)/bptutorial.pl \
	Bio/SeqIO/game.pm \
	$(INST_LIBDIR)/Bio/SeqIO/game.pm \
	Bio/SeqIO/fasta.pm \
	$(INST_LIBDIR)/Bio/SeqIO/fasta.pm \
	Bio/Tools/AlignFactory.pm \
	$(INST_LIBDIR)/Bio/Tools/AlignFactory.pm \
	Bio/SeqIO/game/featureHandler.pm \
	$(INST_LIBDIR)/Bio/SeqIO/game/featureHandler.pm \
	Bio/AlignIO/pfam.pm \
	$(INST_LIBDIR)/Bio/AlignIO/pfam.pm \
	Bio/SeqIO/raw.pm \
	$(INST_LIBDIR)/Bio/SeqIO/raw.pm \
	Bio/Tools/BPbl2seq.pm \
	$(INST_LIBDIR)/Bio/Tools/BPbl2seq.pm \
	Bio/Tools/Blast/HTML.pm \
	$(INST_LIBDIR)/Bio/Tools/Blast/HTML.pm \
	Bio/Root/Utilities.pm \
	$(INST_LIBDIR)/Bio/Root/Utilities.pm \
	Bio/Tools/Blast.pm \
	$(INST_LIBDIR)/Bio/Tools/Blast.pm \
	Bio/Tools/PRFSCAN/Parse.pm \
	$(INST_LIBDIR)/Bio/Tools/PRFSCAN/Parse.pm \
	bioperl.pod \
	$(INST_LIBDIR)/bioperl.pod \
	Bio/SeqFeature/Gene/GeneStructureI.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Gene/GeneStructureI.pm \
	Bio/SeqIO/genbank.pm \
	$(INST_LIBDIR)/Bio/SeqIO/genbank.pm \
	Bio/AlignIO/bl2seq.pm \
	$(INST_LIBDIR)/Bio/AlignIO/bl2seq.pm \
	Bio/Tools/BPlite.pm \
	$(INST_LIBDIR)/Bio/Tools/BPlite.pm \
	Bio/AlignIO/selex.pm \
	$(INST_LIBDIR)/Bio/AlignIO/selex.pm \
	Bio/Range.pm \
	$(INST_LIBDIR)/Bio/Range.pm \
	Bio/DB/SwissProt.pm \
	$(INST_LIBDIR)/Bio/DB/SwissProt.pm \
	Bio/SeqFeature/Gene/TranscriptI.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Gene/TranscriptI.pm \
	Bio/DBLinkContainerI.pm \
	$(INST_LIBDIR)/Bio/DBLinkContainerI.pm \
	Bio/LiveSeq/Range.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Range.pm \
	Bio/SeqFeature/Gene/ExonI.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Gene/ExonI.pm \
	Bio/DB/Ace.pm \
	$(INST_LIBDIR)/Bio/DB/Ace.pm \
	Bio/SeqFeature/Intron.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Intron.pm \
	Bio/DB/RandomAccessI.pm \
	$(INST_LIBDIR)/Bio/DB/RandomAccessI.pm \
	Bio/Index/SwissPfam.pm \
	$(INST_LIBDIR)/Bio/Index/SwissPfam.pm \
	Bio/Location/Fuzzy.pm \
	$(INST_LIBDIR)/Bio/Location/Fuzzy.pm \
	Bio/Tools/Blast/CHANGES \
	$(INST_LIBDIR)/Bio/Tools/Blast/CHANGES \
	Bio/SeqIO/MultiFile.pm \
	$(INST_LIBDIR)/Bio/SeqIO/MultiFile.pm \
	Bio/Tools/RestrictionEnzyme.pm \
	$(INST_LIBDIR)/Bio/Tools/RestrictionEnzyme.pm \
	Bio/Tools/HMMER/Results.pm \
	$(INST_LIBDIR)/Bio/Tools/HMMER/Results.pm \
	Bio/Location/SplitLocationI.pm \
	$(INST_LIBDIR)/Bio/Location/SplitLocationI.pm \
	Bio/Variation/RNAChange.pm \
	$(INST_LIBDIR)/Bio/Variation/RNAChange.pm \
	Bio/LiveSeq/DNA.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/DNA.pm \
	Bio/Tools/Blast/README \
	$(INST_LIBDIR)/Bio/Tools/Blast/README \
	Bio/Seq/LargeSeq.pm \
	$(INST_LIBDIR)/Bio/Seq/LargeSeq.pm \
	Bio/Annotation.pm \
	$(INST_LIBDIR)/Bio/Annotation.pm \
	Bio/LiveSeq/Exon.pm \
	$(INST_LIBDIR)/Bio/LiveSeq/Exon.pm \
	Bio/Location/CoordinatePolicyI.pm \
	$(INST_LIBDIR)/Bio/Location/CoordinatePolicyI.pm \
	Bio/SeqFeature/Similarity.pm \
	$(INST_LIBDIR)/Bio/SeqFeature/Similarity.pm


# --- MakeMaker tool_autosplit section:

# Usage: $(AUTOSPLITFILE) FileToSplit AutoDirToSplitInto
AUTOSPLITFILE = $(PERL) "-I$(PERL_ARCHLIB)" "-I$(PERL_LIB)" -e 'use AutoSplit;autosplit($$ARGV[0], $$ARGV[1], 0, 1, 1) ;'


# --- MakeMaker tool_xsubpp section:


# --- MakeMaker tools_other section:

SHELL = /bin/sh
CHMOD = chmod
CP = cp
LD = gcc
MV = mv
NOOP = $(SHELL) -c true
RM_F = rm -f
RM_RF = rm -rf
TEST_F = test -f
TOUCH = touch
UMASK_NULL = umask 0
DEV_NULL = > /dev/null 2>&1

# The following is a portable way to say mkdir -p
# To see which directories are created, change the if 0 to if 1
MKPATH = $(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -MExtUtils::Command -e mkpath

# This helps us to minimize the effect of the .exists files A yet
# better solution would be to have a stable file in the perl
# distribution with a timestamp of zero. But this solution doesn't
# need any changes to the core distribution and works with older perls
EQUALIZE_TIMESTAMP = $(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -MExtUtils::Command -e eqtime

# Here we warn users that an old packlist file was found somewhere,
# and that they should call some uninstall routine
WARN_IF_OLD_PACKLIST = $(PERL) -we 'exit unless -f $$ARGV[0];' \
-e 'print "WARNING: I have found an old package in\n";' \
-e 'print "\t$$ARGV[0].\n";' \
-e 'print "Please make sure the two installations are not conflicting\n";'

UNINST=0
VERBINST=1

MOD_INSTALL = $(PERL) -I$(INST_LIB) -I$(PERL_LIB) -MExtUtils::Install \
-e "install({@ARGV},'$(VERBINST)',0,'$(UNINST)');"

DOC_INSTALL = $(PERL) -e '$$\="\n\n";' \
-e 'print "=head2 ", scalar(localtime), ": C<", shift, ">", " L<", shift, ">";' \
-e 'print "=over 4";' \
-e 'while (defined($$key = shift) and defined($$val = shift)){print "=item *";print "C<$$key: $$val>";}' \
-e 'print "=back";'

UNINSTALL =   $(PERL) -MExtUtils::Install \
-e 'uninstall($$ARGV[0],1,1); print "\nUninstall is deprecated. Please check the";' \
-e 'print " packlist above carefully.\n  There may be errors. Remove the";' \
-e 'print " appropriate files manually.\n  Sorry for the inconveniences.\n"'


# --- MakeMaker dist section:

DISTVNAME = $(DISTNAME)-$(VERSION)
TAR  = tar
TARFLAGS = cvf
ZIP  = zip
ZIPFLAGS = -r
COMPRESS = gzip -9f
SUFFIX = .gz
SHAR = shar
PREOP = @$(NOOP)
POSTOP = @$(NOOP)
TO_UNIX = @$(NOOP)
CI = ci -u
RCS_LABEL = rcs -Nv$(VERSION_SYM): -q
DIST_CP = best
DIST_DEFAULT = all tardist


# --- MakeMaker macro section:


# --- MakeMaker depend section:


# --- MakeMaker cflags section:


# --- MakeMaker const_loadlibs section:


# --- MakeMaker const_cccmd section:


# --- MakeMaker post_constants section:


# --- MakeMaker pasthru section:

PASTHRU = LIB="$(LIB)"\
	LIBPERL_A="$(LIBPERL_A)"\
	LINKTYPE="$(LINKTYPE)"\
	PREFIX="$(PREFIX)"\
	OPTIMIZE="$(OPTIMIZE)"


# --- MakeMaker c_o section:


# --- MakeMaker xs_c section:


# --- MakeMaker xs_o section:


# --- MakeMaker top_targets section:

#all ::	config $(INST_PM) subdirs linkext manifypods

all :: pure_all htmlifypods manifypods
	@$(NOOP)

pure_all :: config pm_to_blib subdirs linkext
	@$(NOOP)

subdirs :: $(MYEXTLIB)
	@$(NOOP)

config :: Makefile $(INST_LIBDIR)/.exists
	@$(NOOP)

config :: $(INST_ARCHAUTODIR)/.exists
	@$(NOOP)

config :: $(INST_AUTODIR)/.exists
	@$(NOOP)

$(INST_AUTODIR)/.exists :: /usr/lib/perl5/5.6.0/i386-linux/CORE/perl.h
	@$(MKPATH) $(INST_AUTODIR)
	@$(EQUALIZE_TIMESTAMP) /usr/lib/perl5/5.6.0/i386-linux/CORE/perl.h $(INST_AUTODIR)/.exists

	-@$(CHMOD) $(PERM_RWX) $(INST_AUTODIR)

$(INST_LIBDIR)/.exists :: /usr/lib/perl5/5.6.0/i386-linux/CORE/perl.h
	@$(MKPATH) $(INST_LIBDIR)
	@$(EQUALIZE_TIMESTAMP) /usr/lib/perl5/5.6.0/i386-linux/CORE/perl.h $(INST_LIBDIR)/.exists

	-@$(CHMOD) $(PERM_RWX) $(INST_LIBDIR)

$(INST_ARCHAUTODIR)/.exists :: /usr/lib/perl5/5.6.0/i386-linux/CORE/perl.h
	@$(MKPATH) $(INST_ARCHAUTODIR)
	@$(EQUALIZE_TIMESTAMP) /usr/lib/perl5/5.6.0/i386-linux/CORE/perl.h $(INST_ARCHAUTODIR)/.exists

	-@$(CHMOD) $(PERM_RWX) $(INST_ARCHAUTODIR)

config :: $(INST_MAN3DIR)/.exists
	@$(NOOP)


$(INST_MAN3DIR)/.exists :: /usr/lib/perl5/5.6.0/i386-linux/CORE/perl.h
	@$(MKPATH) $(INST_MAN3DIR)
	@$(EQUALIZE_TIMESTAMP) /usr/lib/perl5/5.6.0/i386-linux/CORE/perl.h $(INST_MAN3DIR)/.exists

	-@$(CHMOD) $(PERM_RWX) $(INST_MAN3DIR)

help:
	perldoc ExtUtils::MakeMaker

Version_check:
	@$(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) \
		-MExtUtils::MakeMaker=Version_check \
		-e "Version_check('$(MM_VERSION)')"


# --- MakeMaker linkext section:

linkext :: $(LINKTYPE)
	@$(NOOP)


# --- MakeMaker dlsyms section:


# --- MakeMaker dynamic section:

## $(INST_PM) has been moved to the all: target.
## It remains here for awhile to allow for old usage: "make dynamic"
#dynamic :: Makefile $(INST_DYNAMIC) $(INST_BOOT) $(INST_PM)
dynamic :: Makefile $(INST_DYNAMIC) $(INST_BOOT)
	@$(NOOP)


# --- MakeMaker dynamic_bs section:

BOOTSTRAP =


# --- MakeMaker dynamic_lib section:


# --- MakeMaker static section:

## $(INST_PM) has been moved to the all: target.
## It remains here for awhile to allow for old usage: "make static"
#static :: Makefile $(INST_STATIC) $(INST_PM)
static :: Makefile $(INST_STATIC)
	@$(NOOP)


# --- MakeMaker static_lib section:


# --- MakeMaker htmlifypods section:

htmlifypods : pure_all
	@$(NOOP)


# --- MakeMaker manifypods section:
POD2MAN_EXE = /usr/bin/pod2man
POD2MAN = $(PERL) -we '%m=@ARGV;for (keys %m){' \
-e 'next if -e $$m{$$_} && -M $$m{$$_} < -M $$_ && -M $$m{$$_} < -M "Makefile";' \
-e 'print "Manifying $$m{$$_}\n";' \
-e 'system(qq[$$^X ].q["-I$(PERL_ARCHLIB)" "-I$(PERL_LIB)" $(POD2MAN_EXE) ].qq[$$_>$$m{$$_}])==0 or warn "Couldn\047t install $$m{$$_}\n";' \
-e 'chmod(oct($(PERM_RW))), $$m{$$_} or warn "chmod $(PERM_RW) $$m{$$_}: $$!\n";}'

manifypods : pure_all Bio/Annotation/Reference.pm \
	Bio/Root/Err.pm \
	Bio/Tools/Genscan.pm \
	bioback.pod \
	Bio/AlignIO/meme.pm \
	Bio/AlignIO/fasta.pm \
	Bio/Location/NarrowestCoordPolicy.pm \
	Bio/AlignIO/clustalw.pm \
	Bio/Annotation/DBLink.pm \
	Bio/DB/Fasta.pm \
	Bio/LiveSeq/Intron.pm \
	Bio/AlignIO/mase.pm \
	Bio/LiveSeq/Gene.pm \
	Bio/Tools/PPSEARCH/Parse.pm \
	Bio/Tools/HMMER/Domain.pm \
	Bio/Variation/IO.pm \
	Bio/SeqFeature/Exon.pm \
	Bio/DB/GDB.pm \
	Bio/Tools/Blast/Run/LocalBlast.pm \
	Bio/Variation/AAChange.pm \
	Bio/SeqFeature/Generic.pm \
	Bio/SeqFeature/Gene/GeneStructure.pm \
	Bio/Tools/Blast/Sbjct.pm \
	Bio/Index/EMBL.pm \
	Bio/Tools/Run/RemoteBlast.pm \
	Bio/SeqI.pm \
	Bio/SeqFeature/PolyA.pm \
	Bio/Tools/OddCodes.pm \
	biostart.pod \
	Bio/Root/Xref.pm \
	Bio/Root/Global.pm \
	Bio/Seq.pm \
	Bio/SeqIO/gcg.pm \
	Bio/Tools/Blast/Run/Webblast.pm \
	Bio/Root/Vector.pm \
	Bio/Tools/HMMER/Set.pm \
	Bio/SeqFeature/UTR3.pm \
	Bio/SeqIO/scf.pm \
	Bio/SeqIO/FTHelper.pm \
	Bio/RangeI.pm \
	Bio/LiveSeq/IO/SRS.pm \
	Bio/LiveSeq/Translation.pm \
	Bio/LiveSeq/Chain.pm \
	Bio/PrimarySeq.pm \
	Bio/Tools/Sigcleave.pm \
	Bio/Root/Object.pm \
	Bio/LiveSeq/Mutation.pm \
	Bio/Location/AvWithinCoordPolicy.pm \
	Bio/SeqFeature/Promoter.pm \
	Bio/Tools/IUPAC.pm \
	Bio/SeqIO/ace.pm \
	Bio/LiveSeq/SeqI.pm \
	Bio/Location/FuzzyLocationI.pm \
	Bio/Tools/Genemark.pm \
	Bio/Tools/SeqPattern.pm \
	Bio/SeqFeature/SimilarityPair.pm \
	Bio/SeqIO/pir.pm \
	Bio/Tools/AnalysisResult.pm \
	Bio/Tools/SeqAnal.pm \
	Bio/SeqFeature/Gene/Transcript.pm \
	Bio/Search/Processor/ProcessorI.pm \
	Bio/Seq/RichSeqI.pm \
	Bio/Location/Simple.pm \
	Bio/LiveSeq/IO/Loader.pm \
	Bio/Tools/WWW.pm \
	Bio/Variation/IO/flat.pm \
	Bio/LiveSeq/AARange.pm \
	Bio/Index/AbstractSeq.pm \
	Bio/SeqAnalysisParserI.pm \
	Bio/Tools/GFF.pm \
	Bio/Seq/RichSeq.pm \
	Bio/Tools/ESTScan.pm \
	Bio/Root/RootI.pm \
	Bio/Variation/AAReverseMutate.pm \
	Bio/Variation/IO/xml.pm \
	Bio/DB/EMBL.pm \
	Bio/Tools/BPpsilite.pm \
	Bio/Tools/SeqWords.pm \
	Bio/DB/UpdateableSeqI.pm \
	Bio/SeqFeature/Computation.pm \
	Bio/DB/GenBank.pm \
	Bio/LiveSeq/Prim_Transcript.pm \
	Bio/SeqUtils.pm \
	Bio/Variation/VariantI.pm \
	Bio/LiveSeq/Transcript.pm \
	Bio/LocatableSeq.pm \
	Bio/Tools/Sim4/Exon.pm \
	Bio/Tools/MZEF.pm \
	Bio/Tools/CodonTable.pm \
	Bio/Tools/Run/StandAloneBlast.pm \
	Bio/AlignIO/nexus.pm \
	Bio/Annotation/Comment.pm \
	Bio/Factory/DriverFactory.pm \
	Bio/Tools/PRINTS/Parse.pm \
	Bio/DB/SeqI.pm \
	Bio/AlignIO/prodom.pm \
	Bio/Location/WidestCoordPolicy.pm \
	Bio/Root/IOManager.pm \
	Bio/LiveSeq/Repeat_Region.pm \
	Bio/Factory/SeqAnalysisParserFactoryI.pm \
	Bio/SeqIO/game/seqHandler.pm \
	Bio/LocationI.pm \
	Bio/Tools/Sim4/Results.pm \
	Bio/DB/Universal.pm \
	Bio/Tools/Prediction/Exon.pm \
	Bio/SeqIO/game/idHandler.pm \
	Bio/Search/Result/Fasta.pm \
	Bio/AlignIO/phylip.pm \
	Bio/Tools/Blast/HSP.pm \
	Bio/DB/WebDBSeqI.pm \
	Bio/SeqFeatureI.pm \
	Bio/SeqIO/swiss.pm \
	Bio/Search/Hit/Fasta.pm \
	Bio/Location/Split.pm \
	Bio/Tools/Grail.pm \
	Bio/DB/GenPept.pm \
	Bio/Tools/SeqStats.pm \
	Bio/LiveSeq/ChainI.pm \
	Bio/Variation/SeqDiff.pm \
	Bio/PrimarySeqI.pm \
	Bio/Variation/DNAMutation.pm \
	Bio/Search/Result/ResultI.pm \
	Bio/Search/Processor.pm \
	Bio/Tools/Prediction/Gene.pm \
	Bio/UnivAln.pm \
	Bio/Variation/Allele.pm \
	Bio/SeqFeature/FeaturePair.pm \
	biodesign.pod \
	Bio/AlignIO/msf.pm \
	Bio/SeqIO/embl.pm \
	Bio/LiveSeq/Repeat_Unit.pm \
	Bio/Tools/Fasta.pm \
	Bio/SeqFeature/Gene/Exon.pm \
	Bio/Root/IO.pm \
	Bio/Search/Hit/HitI.pm \
	Bio/DB/NCBIHelper.pm \
	Bio/Tools/Run/Alignment/TCoffee.pm \
	Bio/SeqIO/largefasta.pm \
	Bio/Index/Swissprot.pm \
	Bio/Tools/Run/Alignment/Clustalw.pm \
	Bio/Species.pm \
	Bio/Search/Processor/Fasta.pm \
	Bio/Index/Abstract.pm \
	Bio/Seq/LargePrimarySeq.pm \
	Bio/LiveSeq/Mutator.pm \
	Bio/AlignIO.pm \
	Bio/Tools/pSW.pm \
	Bio/Tools/BPlite/Iteration.pm \
	Bio/Factory/SeqAnalysisParserFactory.pm \
	Bio/SimpleAlign.pm \
	Bio/SeqFeature/UTR5.pm \
	Bio/Index/GenBank.pm \
	Bio/AlignIO/stockholm.pm \
	Bio/SeqIO.pm \
	Bio/Index/Fasta.pm \
	Bio/SearchDist.pm \
	Bio/LiveSeq/IO/BioPerl.pm \
	bptutorial.pl \
	Bio/SeqIO/game.pm \
	Bio/SeqIO/fasta.pm \
	Bio/Tools/AlignFactory.pm \
	Bio/SeqIO/game/featureHandler.pm \
	Bio/AlignIO/pfam.pm \
	Bio/SeqIO/raw.pm \
	Bio/Tools/BPbl2seq.pm \
	Bio/Tools/Blast/HTML.pm \
	Bio/Root/Utilities.pm \
	Bio/Tools/Blast.pm \
	Bio/Tools/PRFSCAN/Parse.pm \
	bioperl.pod \
	Bio/SeqFeature/Gene/GeneStructureI.pm \
	Bio/SeqIO/genbank.pm \
	Bio/AlignIO/bl2seq.pm \
	Bio/Tools/BPlite.pm \
	Bio/AlignIO/selex.pm \
	Bio/Range.pm \
	Bio/DB/SwissProt.pm \
	Bio/SeqFeature/Gene/TranscriptI.pm \
	Bio/DBLinkContainerI.pm \
	Bio/LiveSeq/Range.pm \
	Bio/SeqFeature/Gene/ExonI.pm \
	Bio/DB/Ace.pm \
	Bio/SeqFeature/Intron.pm \
	Bio/DB/RandomAccessI.pm \
	Bio/Index/SwissPfam.pm \
	Bio/Location/Fuzzy.pm \
	Bio/SeqIO/MultiFile.pm \
	Bio/Tools/RestrictionEnzyme.pm \
	Bio/Tools/HMMER/Results.pm \
	Bio/Location/SplitLocationI.pm \
	Bio/Variation/RNAChange.pm \
	Bio/LiveSeq/DNA.pm \
	Bio/Seq/LargeSeq.pm \
	Bio/Annotation.pm \
	Bio/LiveSeq/Exon.pm \
	Bio/Location/CoordinatePolicyI.pm \
	Bio/SeqFeature/Similarity.pm
	@$(POD2MAN) \
	Bio/Annotation/Reference.pm \
	$(INST_MAN3DIR)/Bio::Annotation::Reference.$(MAN3EXT) \
	Bio/Root/Err.pm \
	$(INST_MAN3DIR)/Bio::Root::Err.$(MAN3EXT) \
	Bio/Tools/Genscan.pm \
	$(INST_MAN3DIR)/Bio::Tools::Genscan.$(MAN3EXT) \
	bioback.pod \
	$(INST_MAN3DIR)/bioback.$(MAN3EXT) \
	Bio/AlignIO/meme.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::meme.$(MAN3EXT) \
	Bio/AlignIO/fasta.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::fasta.$(MAN3EXT) \
	Bio/Location/NarrowestCoordPolicy.pm \
	$(INST_MAN3DIR)/Bio::Location::NarrowestCoordPolicy.$(MAN3EXT) \
	Bio/AlignIO/clustalw.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::clustalw.$(MAN3EXT) \
	Bio/Annotation/DBLink.pm \
	$(INST_MAN3DIR)/Bio::Annotation::DBLink.$(MAN3EXT) \
	Bio/DB/Fasta.pm \
	$(INST_MAN3DIR)/Bio::DB::Fasta.$(MAN3EXT) \
	Bio/LiveSeq/Intron.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Intron.$(MAN3EXT) \
	Bio/AlignIO/mase.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::mase.$(MAN3EXT) \
	Bio/LiveSeq/Gene.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Gene.$(MAN3EXT) \
	Bio/Tools/PPSEARCH/Parse.pm \
	$(INST_MAN3DIR)/Bio::Tools::PPSEARCH::Parse.$(MAN3EXT) \
	Bio/Tools/HMMER/Domain.pm \
	$(INST_MAN3DIR)/Bio::Tools::HMMER::Domain.$(MAN3EXT) \
	Bio/Variation/IO.pm \
	$(INST_MAN3DIR)/Bio::Variation::IO.$(MAN3EXT) \
	Bio/SeqFeature/Exon.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Exon.$(MAN3EXT) \
	Bio/DB/GDB.pm \
	$(INST_MAN3DIR)/Bio::DB::GDB.$(MAN3EXT) \
	Bio/Tools/Blast/Run/LocalBlast.pm \
	$(INST_MAN3DIR)/Bio::Tools::Blast::Run::LocalBlast.$(MAN3EXT) \
	Bio/Variation/AAChange.pm \
	$(INST_MAN3DIR)/Bio::Variation::AAChange.$(MAN3EXT) \
	Bio/SeqFeature/Generic.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Generic.$(MAN3EXT) \
	Bio/SeqFeature/Gene/GeneStructure.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Gene::GeneStructure.$(MAN3EXT) \
	Bio/Tools/Blast/Sbjct.pm \
	$(INST_MAN3DIR)/Bio::Tools::Blast::Sbjct.$(MAN3EXT) \
	Bio/Index/EMBL.pm \
	$(INST_MAN3DIR)/Bio::Index::EMBL.$(MAN3EXT) \
	Bio/Tools/Run/RemoteBlast.pm \
	$(INST_MAN3DIR)/Bio::Tools::Run::RemoteBlast.$(MAN3EXT) \
	Bio/SeqI.pm \
	$(INST_MAN3DIR)/Bio::SeqI.$(MAN3EXT) \
	Bio/SeqFeature/PolyA.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::PolyA.$(MAN3EXT) \
	Bio/Tools/OddCodes.pm \
	$(INST_MAN3DIR)/Bio::Tools::OddCodes.$(MAN3EXT) \
	biostart.pod \
	$(INST_MAN3DIR)/biostart.$(MAN3EXT) \
	Bio/Root/Xref.pm \
	$(INST_MAN3DIR)/Bio::Root::Xref.$(MAN3EXT) \
	Bio/Root/Global.pm \
	$(INST_MAN3DIR)/Bio::Root::Global.$(MAN3EXT) \
	Bio/Seq.pm \
	$(INST_MAN3DIR)/Bio::Seq.$(MAN3EXT) \
	Bio/SeqIO/gcg.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::gcg.$(MAN3EXT) \
	Bio/Tools/Blast/Run/Webblast.pm \
	$(INST_MAN3DIR)/Bio::Tools::Blast::Run::Webblast.$(MAN3EXT) \
	Bio/Root/Vector.pm \
	$(INST_MAN3DIR)/Bio::Root::Vector.$(MAN3EXT) \
	Bio/Tools/HMMER/Set.pm \
	$(INST_MAN3DIR)/Bio::Tools::HMMER::Set.$(MAN3EXT) \
	Bio/SeqFeature/UTR3.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::UTR3.$(MAN3EXT) \
	Bio/SeqIO/scf.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::scf.$(MAN3EXT) \
	Bio/SeqIO/FTHelper.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::FTHelper.$(MAN3EXT) \
	Bio/RangeI.pm \
	$(INST_MAN3DIR)/Bio::RangeI.$(MAN3EXT) \
	Bio/LiveSeq/IO/SRS.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::IO::SRS.$(MAN3EXT) \
	Bio/LiveSeq/Translation.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Translation.$(MAN3EXT) \
	Bio/LiveSeq/Chain.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Chain.$(MAN3EXT) \
	Bio/PrimarySeq.pm \
	$(INST_MAN3DIR)/Bio::PrimarySeq.$(MAN3EXT) \
	Bio/Tools/Sigcleave.pm \
	$(INST_MAN3DIR)/Bio::Tools::Sigcleave.$(MAN3EXT) \
	Bio/Root/Object.pm \
	$(INST_MAN3DIR)/Bio::Root::Object.$(MAN3EXT) \
	Bio/LiveSeq/Mutation.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Mutation.$(MAN3EXT) \
	Bio/Location/AvWithinCoordPolicy.pm \
	$(INST_MAN3DIR)/Bio::Location::AvWithinCoordPolicy.$(MAN3EXT) \
	Bio/SeqFeature/Promoter.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Promoter.$(MAN3EXT) \
	Bio/Tools/IUPAC.pm \
	$(INST_MAN3DIR)/Bio::Tools::IUPAC.$(MAN3EXT) \
	Bio/SeqIO/ace.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::ace.$(MAN3EXT) \
	Bio/LiveSeq/SeqI.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::SeqI.$(MAN3EXT) \
	Bio/Location/FuzzyLocationI.pm \
	$(INST_MAN3DIR)/Bio::Location::FuzzyLocationI.$(MAN3EXT) \
	Bio/Tools/Genemark.pm \
	$(INST_MAN3DIR)/Bio::Tools::Genemark.$(MAN3EXT) \
	Bio/Tools/SeqPattern.pm \
	$(INST_MAN3DIR)/Bio::Tools::SeqPattern.$(MAN3EXT) \
	Bio/SeqFeature/SimilarityPair.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::SimilarityPair.$(MAN3EXT) \
	Bio/SeqIO/pir.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::pir.$(MAN3EXT) \
	Bio/Tools/AnalysisResult.pm \
	$(INST_MAN3DIR)/Bio::Tools::AnalysisResult.$(MAN3EXT) \
	Bio/Tools/SeqAnal.pm \
	$(INST_MAN3DIR)/Bio::Tools::SeqAnal.$(MAN3EXT) \
	Bio/SeqFeature/Gene/Transcript.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Gene::Transcript.$(MAN3EXT) \
	Bio/Search/Processor/ProcessorI.pm \
	$(INST_MAN3DIR)/Bio::Search::Processor::ProcessorI.$(MAN3EXT) \
	Bio/Seq/RichSeqI.pm \
	$(INST_MAN3DIR)/Bio::Seq::RichSeqI.$(MAN3EXT) \
	Bio/Location/Simple.pm \
	$(INST_MAN3DIR)/Bio::Location::Simple.$(MAN3EXT) \
	Bio/LiveSeq/IO/Loader.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::IO::Loader.$(MAN3EXT) \
	Bio/Tools/WWW.pm \
	$(INST_MAN3DIR)/Bio::Tools::WWW.$(MAN3EXT) \
	Bio/Variation/IO/flat.pm \
	$(INST_MAN3DIR)/Bio::Variation::IO::flat.$(MAN3EXT) \
	Bio/LiveSeq/AARange.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::AARange.$(MAN3EXT) \
	Bio/Index/AbstractSeq.pm \
	$(INST_MAN3DIR)/Bio::Index::AbstractSeq.$(MAN3EXT) \
	Bio/SeqAnalysisParserI.pm \
	$(INST_MAN3DIR)/Bio::SeqAnalysisParserI.$(MAN3EXT) \
	Bio/Tools/GFF.pm \
	$(INST_MAN3DIR)/Bio::Tools::GFF.$(MAN3EXT) \
	Bio/Seq/RichSeq.pm \
	$(INST_MAN3DIR)/Bio::Seq::RichSeq.$(MAN3EXT) \
	Bio/Tools/ESTScan.pm \
	$(INST_MAN3DIR)/Bio::Tools::ESTScan.$(MAN3EXT) \
	Bio/Root/RootI.pm \
	$(INST_MAN3DIR)/Bio::Root::RootI.$(MAN3EXT) \
	Bio/Variation/AAReverseMutate.pm \
	$(INST_MAN3DIR)/Bio::Variation::AAReverseMutate.$(MAN3EXT) \
	Bio/Variation/IO/xml.pm \
	$(INST_MAN3DIR)/Bio::Variation::IO::xml.$(MAN3EXT) \
	Bio/DB/EMBL.pm \
	$(INST_MAN3DIR)/Bio::DB::EMBL.$(MAN3EXT) \
	Bio/Tools/BPpsilite.pm \
	$(INST_MAN3DIR)/Bio::Tools::BPpsilite.$(MAN3EXT) \
	Bio/Tools/SeqWords.pm \
	$(INST_MAN3DIR)/Bio::Tools::SeqWords.$(MAN3EXT) \
	Bio/DB/UpdateableSeqI.pm \
	$(INST_MAN3DIR)/Bio::DB::UpdateableSeqI.$(MAN3EXT) \
	Bio/SeqFeature/Computation.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Computation.$(MAN3EXT) \
	Bio/DB/GenBank.pm \
	$(INST_MAN3DIR)/Bio::DB::GenBank.$(MAN3EXT) \
	Bio/LiveSeq/Prim_Transcript.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Prim_Transcript.$(MAN3EXT) \
	Bio/SeqUtils.pm \
	$(INST_MAN3DIR)/Bio::SeqUtils.$(MAN3EXT) \
	Bio/Variation/VariantI.pm \
	$(INST_MAN3DIR)/Bio::Variation::VariantI.$(MAN3EXT) \
	Bio/LiveSeq/Transcript.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Transcript.$(MAN3EXT) \
	Bio/LocatableSeq.pm \
	$(INST_MAN3DIR)/Bio::LocatableSeq.$(MAN3EXT) \
	Bio/Tools/Sim4/Exon.pm \
	$(INST_MAN3DIR)/Bio::Tools::Sim4::Exon.$(MAN3EXT) \
	Bio/Tools/MZEF.pm \
	$(INST_MAN3DIR)/Bio::Tools::MZEF.$(MAN3EXT) \
	Bio/Tools/CodonTable.pm \
	$(INST_MAN3DIR)/Bio::Tools::CodonTable.$(MAN3EXT) \
	Bio/Tools/Run/StandAloneBlast.pm \
	$(INST_MAN3DIR)/Bio::Tools::Run::StandAloneBlast.$(MAN3EXT) \
	Bio/AlignIO/nexus.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::nexus.$(MAN3EXT) \
	Bio/Annotation/Comment.pm \
	$(INST_MAN3DIR)/Bio::Annotation::Comment.$(MAN3EXT) \
	Bio/Factory/DriverFactory.pm \
	$(INST_MAN3DIR)/Bio::Factory::DriverFactory.$(MAN3EXT) \
	Bio/Tools/PRINTS/Parse.pm \
	$(INST_MAN3DIR)/Bio::Tools::PRINTS::Parse.$(MAN3EXT) \
	Bio/DB/SeqI.pm \
	$(INST_MAN3DIR)/Bio::DB::SeqI.$(MAN3EXT) \
	Bio/AlignIO/prodom.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::prodom.$(MAN3EXT) \
	Bio/Location/WidestCoordPolicy.pm \
	$(INST_MAN3DIR)/Bio::Location::WidestCoordPolicy.$(MAN3EXT) \
	Bio/Root/IOManager.pm \
	$(INST_MAN3DIR)/Bio::Root::IOManager.$(MAN3EXT) \
	Bio/LiveSeq/Repeat_Region.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Repeat_Region.$(MAN3EXT) \
	Bio/Factory/SeqAnalysisParserFactoryI.pm \
	$(INST_MAN3DIR)/Bio::Factory::SeqAnalysisParserFactoryI.$(MAN3EXT) \
	Bio/SeqIO/game/seqHandler.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::game::seqHandler.$(MAN3EXT) \
	Bio/LocationI.pm \
	$(INST_MAN3DIR)/Bio::LocationI.$(MAN3EXT) \
	Bio/Tools/Sim4/Results.pm \
	$(INST_MAN3DIR)/Bio::Tools::Sim4::Results.$(MAN3EXT) \
	Bio/DB/Universal.pm \
	$(INST_MAN3DIR)/Bio::DB::Universal.$(MAN3EXT) \
	Bio/Tools/Prediction/Exon.pm \
	$(INST_MAN3DIR)/Bio::Tools::Prediction::Exon.$(MAN3EXT) \
	Bio/SeqIO/game/idHandler.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::game::idHandler.$(MAN3EXT) \
	Bio/Search/Result/Fasta.pm \
	$(INST_MAN3DIR)/Bio::Search::Result::Fasta.$(MAN3EXT) \
	Bio/AlignIO/phylip.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::phylip.$(MAN3EXT) \
	Bio/Tools/Blast/HSP.pm \
	$(INST_MAN3DIR)/Bio::Tools::Blast::HSP.$(MAN3EXT) \
	Bio/DB/WebDBSeqI.pm \
	$(INST_MAN3DIR)/Bio::DB::WebDBSeqI.$(MAN3EXT) \
	Bio/SeqFeatureI.pm \
	$(INST_MAN3DIR)/Bio::SeqFeatureI.$(MAN3EXT) \
	Bio/SeqIO/swiss.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::swiss.$(MAN3EXT) \
	Bio/Search/Hit/Fasta.pm \
	$(INST_MAN3DIR)/Bio::Search::Hit::Fasta.$(MAN3EXT) \
	Bio/Location/Split.pm \
	$(INST_MAN3DIR)/Bio::Location::Split.$(MAN3EXT) \
	Bio/Tools/Grail.pm \
	$(INST_MAN3DIR)/Bio::Tools::Grail.$(MAN3EXT) \
	Bio/DB/GenPept.pm \
	$(INST_MAN3DIR)/Bio::DB::GenPept.$(MAN3EXT) \
	Bio/Tools/SeqStats.pm \
	$(INST_MAN3DIR)/Bio::Tools::SeqStats.$(MAN3EXT) \
	Bio/LiveSeq/ChainI.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::ChainI.$(MAN3EXT) \
	Bio/Variation/SeqDiff.pm \
	$(INST_MAN3DIR)/Bio::Variation::SeqDiff.$(MAN3EXT) \
	Bio/PrimarySeqI.pm \
	$(INST_MAN3DIR)/Bio::PrimarySeqI.$(MAN3EXT) \
	Bio/Variation/DNAMutation.pm \
	$(INST_MAN3DIR)/Bio::Variation::DNAMutation.$(MAN3EXT) \
	Bio/Search/Result/ResultI.pm \
	$(INST_MAN3DIR)/Bio::Search::Result::ResultI.$(MAN3EXT) \
	Bio/Search/Processor.pm \
	$(INST_MAN3DIR)/Bio::Search::Processor.$(MAN3EXT) \
	Bio/Tools/Prediction/Gene.pm \
	$(INST_MAN3DIR)/Bio::Tools::Prediction::Gene.$(MAN3EXT) \
	Bio/UnivAln.pm \
	$(INST_MAN3DIR)/Bio::UnivAln.$(MAN3EXT) \
	Bio/Variation/Allele.pm \
	$(INST_MAN3DIR)/Bio::Variation::Allele.$(MAN3EXT) \
	Bio/SeqFeature/FeaturePair.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::FeaturePair.$(MAN3EXT) \
	biodesign.pod \
	$(INST_MAN3DIR)/biodesign.$(MAN3EXT) \
	Bio/AlignIO/msf.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::msf.$(MAN3EXT) \
	Bio/SeqIO/embl.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::embl.$(MAN3EXT) \
	Bio/LiveSeq/Repeat_Unit.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Repeat_Unit.$(MAN3EXT) \
	Bio/Tools/Fasta.pm \
	$(INST_MAN3DIR)/Bio::Tools::Fasta.$(MAN3EXT) \
	Bio/SeqFeature/Gene/Exon.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Gene::Exon.$(MAN3EXT) \
	Bio/Root/IO.pm \
	$(INST_MAN3DIR)/Bio::Root::IO.$(MAN3EXT) \
	Bio/Search/Hit/HitI.pm \
	$(INST_MAN3DIR)/Bio::Search::Hit::HitI.$(MAN3EXT) \
	Bio/DB/NCBIHelper.pm \
	$(INST_MAN3DIR)/Bio::DB::NCBIHelper.$(MAN3EXT) \
	Bio/Tools/Run/Alignment/TCoffee.pm \
	$(INST_MAN3DIR)/Bio::Tools::Run::Alignment::TCoffee.$(MAN3EXT) \
	Bio/SeqIO/largefasta.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::largefasta.$(MAN3EXT) \
	Bio/Index/Swissprot.pm \
	$(INST_MAN3DIR)/Bio::Index::Swissprot.$(MAN3EXT) \
	Bio/Tools/Run/Alignment/Clustalw.pm \
	$(INST_MAN3DIR)/Bio::Tools::Run::Alignment::Clustalw.$(MAN3EXT) \
	Bio/Species.pm \
	$(INST_MAN3DIR)/Bio::Species.$(MAN3EXT) \
	Bio/Search/Processor/Fasta.pm \
	$(INST_MAN3DIR)/Bio::Search::Processor::Fasta.$(MAN3EXT) \
	Bio/Index/Abstract.pm \
	$(INST_MAN3DIR)/Bio::Index::Abstract.$(MAN3EXT) \
	Bio/Seq/LargePrimarySeq.pm \
	$(INST_MAN3DIR)/Bio::Seq::LargePrimarySeq.$(MAN3EXT) \
	Bio/LiveSeq/Mutator.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Mutator.$(MAN3EXT) \
	Bio/AlignIO.pm \
	$(INST_MAN3DIR)/Bio::AlignIO.$(MAN3EXT) \
	Bio/Tools/pSW.pm \
	$(INST_MAN3DIR)/Bio::Tools::pSW.$(MAN3EXT) \
	Bio/Tools/BPlite/Iteration.pm \
	$(INST_MAN3DIR)/Bio::Tools::BPlite::Iteration.$(MAN3EXT) \
	Bio/Factory/SeqAnalysisParserFactory.pm \
	$(INST_MAN3DIR)/Bio::Factory::SeqAnalysisParserFactory.$(MAN3EXT) \
	Bio/SimpleAlign.pm \
	$(INST_MAN3DIR)/Bio::SimpleAlign.$(MAN3EXT) \
	Bio/SeqFeature/UTR5.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::UTR5.$(MAN3EXT) \
	Bio/Index/GenBank.pm \
	$(INST_MAN3DIR)/Bio::Index::GenBank.$(MAN3EXT) \
	Bio/AlignIO/stockholm.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::stockholm.$(MAN3EXT) \
	Bio/SeqIO.pm \
	$(INST_MAN3DIR)/Bio::SeqIO.$(MAN3EXT) \
	Bio/Index/Fasta.pm \
	$(INST_MAN3DIR)/Bio::Index::Fasta.$(MAN3EXT) \
	Bio/SearchDist.pm \
	$(INST_MAN3DIR)/Bio::SearchDist.$(MAN3EXT) \
	Bio/LiveSeq/IO/BioPerl.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::IO::BioPerl.$(MAN3EXT) \
	bptutorial.pl \
	$(INST_MAN3DIR)/bptutorial.$(MAN3EXT) \
	Bio/SeqIO/game.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::game.$(MAN3EXT) \
	Bio/SeqIO/fasta.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::fasta.$(MAN3EXT) \
	Bio/Tools/AlignFactory.pm \
	$(INST_MAN3DIR)/Bio::Tools::AlignFactory.$(MAN3EXT) \
	Bio/SeqIO/game/featureHandler.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::game::featureHandler.$(MAN3EXT) \
	Bio/AlignIO/pfam.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::pfam.$(MAN3EXT) \
	Bio/SeqIO/raw.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::raw.$(MAN3EXT) \
	Bio/Tools/BPbl2seq.pm \
	$(INST_MAN3DIR)/Bio::Tools::BPbl2seq.$(MAN3EXT) \
	Bio/Tools/Blast/HTML.pm \
	$(INST_MAN3DIR)/Bio::Tools::Blast::HTML.$(MAN3EXT) \
	Bio/Root/Utilities.pm \
	$(INST_MAN3DIR)/Bio::Root::Utilities.$(MAN3EXT) \
	Bio/Tools/Blast.pm \
	$(INST_MAN3DIR)/Bio::Tools::Blast.$(MAN3EXT) \
	Bio/Tools/PRFSCAN/Parse.pm \
	$(INST_MAN3DIR)/Bio::Tools::PRFSCAN::Parse.$(MAN3EXT) \
	bioperl.pod \
	$(INST_MAN3DIR)/bioperl.$(MAN3EXT) \
	Bio/SeqFeature/Gene/GeneStructureI.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Gene::GeneStructureI.$(MAN3EXT) \
	Bio/SeqIO/genbank.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::genbank.$(MAN3EXT) \
	Bio/AlignIO/bl2seq.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::bl2seq.$(MAN3EXT) \
	Bio/Tools/BPlite.pm \
	$(INST_MAN3DIR)/Bio::Tools::BPlite.$(MAN3EXT) \
	Bio/AlignIO/selex.pm \
	$(INST_MAN3DIR)/Bio::AlignIO::selex.$(MAN3EXT) \
	Bio/Range.pm \
	$(INST_MAN3DIR)/Bio::Range.$(MAN3EXT) \
	Bio/DB/SwissProt.pm \
	$(INST_MAN3DIR)/Bio::DB::SwissProt.$(MAN3EXT) \
	Bio/SeqFeature/Gene/TranscriptI.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Gene::TranscriptI.$(MAN3EXT) \
	Bio/DBLinkContainerI.pm \
	$(INST_MAN3DIR)/Bio::DBLinkContainerI.$(MAN3EXT) \
	Bio/LiveSeq/Range.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Range.$(MAN3EXT) \
	Bio/SeqFeature/Gene/ExonI.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Gene::ExonI.$(MAN3EXT) \
	Bio/DB/Ace.pm \
	$(INST_MAN3DIR)/Bio::DB::Ace.$(MAN3EXT) \
	Bio/SeqFeature/Intron.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Intron.$(MAN3EXT) \
	Bio/DB/RandomAccessI.pm \
	$(INST_MAN3DIR)/Bio::DB::RandomAccessI.$(MAN3EXT) \
	Bio/Index/SwissPfam.pm \
	$(INST_MAN3DIR)/Bio::Index::SwissPfam.$(MAN3EXT) \
	Bio/Location/Fuzzy.pm \
	$(INST_MAN3DIR)/Bio::Location::Fuzzy.$(MAN3EXT) \
	Bio/SeqIO/MultiFile.pm \
	$(INST_MAN3DIR)/Bio::SeqIO::MultiFile.$(MAN3EXT) \
	Bio/Tools/RestrictionEnzyme.pm \
	$(INST_MAN3DIR)/Bio::Tools::RestrictionEnzyme.$(MAN3EXT) \
	Bio/Tools/HMMER/Results.pm \
	$(INST_MAN3DIR)/Bio::Tools::HMMER::Results.$(MAN3EXT) \
	Bio/Location/SplitLocationI.pm \
	$(INST_MAN3DIR)/Bio::Location::SplitLocationI.$(MAN3EXT) \
	Bio/Variation/RNAChange.pm \
	$(INST_MAN3DIR)/Bio::Variation::RNAChange.$(MAN3EXT) \
	Bio/LiveSeq/DNA.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::DNA.$(MAN3EXT) \
	Bio/Seq/LargeSeq.pm \
	$(INST_MAN3DIR)/Bio::Seq::LargeSeq.$(MAN3EXT) \
	Bio/Annotation.pm \
	$(INST_MAN3DIR)/Bio::Annotation.$(MAN3EXT) \
	Bio/LiveSeq/Exon.pm \
	$(INST_MAN3DIR)/Bio::LiveSeq::Exon.$(MAN3EXT) \
	Bio/Location/CoordinatePolicyI.pm \
	$(INST_MAN3DIR)/Bio::Location::CoordinatePolicyI.$(MAN3EXT) \
	Bio/SeqFeature/Similarity.pm \
	$(INST_MAN3DIR)/Bio::SeqFeature::Similarity.$(MAN3EXT)

# --- MakeMaker processPL section:


# --- MakeMaker installbin section:


# --- MakeMaker subdirs section:

# none

# --- MakeMaker clean section:

# Delete temporary files but do not touch installed files. We don't delete
# the Makefile here so a later make realclean still has a makefile to use.

clean ::
	-rm -rf ./blib $(MAKE_APERL_FILE) $(INST_ARCHAUTODIR)/extralibs.all perlmain.c mon.out core core.*perl.*.? *perl.core so_locations pm_to_blib *~ */*~ */*/*~ *$(OBJ_EXT) *$(LIB_EXT) perl.exe $(BOOTSTRAP) $(BASEEXT).bso $(BASEEXT).def $(BASEEXT).exp
	-mv Makefile Makefile.old $(DEV_NULL)


# --- MakeMaker realclean section:

# Delete temporary files (via clean) and also delete installed files
realclean purge ::  clean
	rm -rf $(INST_AUTODIR) $(INST_ARCHAUTODIR)
	rm -f $(INST_LIBDIR)/Bio/Annotation/Reference.pm $(INST_LIBDIR)/Bio/Root/Err.pm $(INST_LIBDIR)/Bio/Tools/Genscan.pm $(INST_LIBDIR)/bioback.pod $(INST_LIBDIR)/Bio/AlignIO/meme.pm $(INST_LIBDIR)/Bio/AlignIO/fasta.pm $(INST_LIBDIR)/Bio/Location/NarrowestCoordPolicy.pm $(INST_LIBDIR)/Bio/AlignIO/clustalw.pm $(INST_LIBDIR)/Bio/Tools/Blast/Run/postclient.pl $(INST_LIBDIR)/Bio/Annotation/DBLink.pm $(INST_LIBDIR)/Bio/DB/Fasta.pm $(INST_LIBDIR)/Bio/LiveSeq/Intron.pm $(INST_LIBDIR)/Bio/AlignIO/mase.pm $(INST_LIBDIR)/Bio/LiveSeq/Gene.pm $(INST_LIBDIR)/Bio/Tools/PPSEARCH/Parse.pm $(INST_LIBDIR)/Bio/Tools/HMMER/Domain.pm $(INST_LIBDIR)/Bio/Variation/IO.pm $(INST_LIBDIR)/Bio/SeqFeature/Exon.pm $(INST_LIBDIR)/Bio/DB/GDB.pm $(INST_LIBDIR)/Bio/Tools/Blast/Run/LocalBlast.pm $(INST_LIBDIR)/Bio/Variation/AAChange.pm $(INST_LIBDIR)/Bio/SeqFeature/Generic.pm $(INST_LIBDIR)/Bio/SeqFeature/Gene/GeneStructure.pm $(INST_LIBDIR)/Bio/Tools/Blast/Sbjct.pm $(INST_LIBDIR)/Bio/Index/EMBL.pm $(INST_LIBDIR)/Bio/Tools/Run/RemoteBlast.pm $(INST_LIBDIR)/Bio/SeqI.pm $(INST_LIBDIR)/Bio/SeqFeature/PolyA.pm $(INST_LIBDIR)/Bio/Tools/OddCodes.pm $(INST_LIBDIR)/biostart.pod $(INST_LIBDIR)/Bio/Root/Xref.pm $(INST_LIBDIR)/Bio/Root/Global.pm $(INST_LIBDIR)/Bio/Seq.pm $(INST_LIBDIR)/Bio/SeqIO/gcg.pm $(INST_LIBDIR)/Bio/Tools/Blast/Run/Webblast.pm $(INST_LIBDIR)/Bio/Root/Vector.pm $(INST_LIBDIR)/Bio/Tools/HMMER/Set.pm $(INST_LIBDIR)/Bio/SeqFeature/UTR3.pm $(INST_LIBDIR)/Bio/SeqIO/scf.pm $(INST_LIBDIR)/Bio/SeqIO/FTHelper.pm $(INST_LIBDIR)/Bio/RangeI.pm $(INST_LIBDIR)/Bio/LiveSeq/IO/SRS.pm $(INST_LIBDIR)/Bio/LiveSeq/Translation.pm $(INST_LIBDIR)/Bio/LiveSeq/Chain.pm $(INST_LIBDIR)/Bio/PrimarySeq.pm $(INST_LIBDIR)/Bio/Tools/BPlite/HSP.pm $(INST_LIBDIR)/Bio/Tools/Sigcleave.pm $(INST_LIBDIR)/Bio/Root/Object.pm $(INST_LIBDIR)/Bio/LiveSeq/Mutation.pm $(INST_LIBDIR)/Bio/Location/AvWithinCoordPolicy.pm $(INST_LIBDIR)/Bio/SeqFeature/Promoter.pm $(INST_LIBDIR)/Bio/Tools/IUPAC.pm $(INST_LIBDIR)/Bio/SeqIO/ace.pm $(INST_LIBDIR)/Bio/LiveSeq/SeqI.pm $(INST_LIBDIR)/Bio/Location/FuzzyLocationI.pm $(INST_LIBDIR)/Bio/Tools/Blast/Run/Makefile.PL $(INST_LIBDIR)/Bio/Tools/Genemark.pm $(INST_LIBDIR)/Bio/Tools/SeqPattern.pm $(INST_LIBDIR)/Bio/SeqFeature/SimilarityPair.pm $(INST_LIBDIR)/Bio/SeqIO/pir.pm $(INST_LIBDIR)/Bio/Tools/AnalysisResult.pm $(INST_LIBDIR)/Bio/Tools/SeqAnal.pm $(INST_LIBDIR)/Bio/SeqFeature/Gene/Transcript.pm $(INST_LIBDIR)/Bio/Search/Processor/ProcessorI.pm $(INST_LIBDIR)/Bio/Seq/RichSeqI.pm $(INST_LIBDIR)/Bio/Location/Simple.pm $(INST_LIBDIR)/Bio/LiveSeq/IO/Loader.pm $(INST_LIBDIR)/Bio/Tools/WWW.pm $(INST_LIBDIR)/Bio/Variation/IO/flat.pm $(INST_LIBDIR)/Bio/LiveSeq/AARange.pm $(INST_LIBDIR)/Bio/Index/AbstractSeq.pm $(INST_LIBDIR)/Bio/SeqAnalysisParserI.pm $(INST_LIBDIR)/Bio/Tools/GFF.pm $(INST_LIBDIR)/Bio/Seq/RichSeq.pm $(INST_LIBDIR)/Bio/Tools/ESTScan.pm $(INST_LIBDIR)/Bio/Root/RootI.pm $(INST_LIBDIR)/Bio/Variation/AAReverseMutate.pm $(INST_LIBDIR)/Bio/Variation/IO/xml.pm $(INST_LIBDIR)/Bio/DB/EMBL.pm $(INST_LIBDIR)/Bio/Tools/BPpsilite.pm $(INST_LIBDIR)/Bio/Tools/SeqWords.pm $(INST_LIBDIR)/Bio/DB/UpdateableSeqI.pm $(INST_LIBDIR)/Bio/SeqFeature/Computation.pm $(INST_LIBDIR)/Bio/DB/GenBank.pm $(INST_LIBDIR)/Bio/LiveSeq/Prim_Transcript.pm $(INST_LIBDIR)/Bio/SeqUtils.pm $(INST_LIBDIR)/Bio/Variation/VariantI.pm $(INST_LIBDIR)/Bio/LiveSeq/Transcript.pm $(INST_LIBDIR)/Bio/LocatableSeq.pm $(INST_LIBDIR)/Bio/Tools/Sim4/Exon.pm $(INST_LIBDIR)/Bio/Tools/MZEF.pm $(INST_LIBDIR)/Bio/Tools/CodonTable.pm $(INST_LIBDIR)/Bio/Tools/Run/StandAloneBlast.pm $(INST_LIBDIR)/Bio/Variation/README $(INST_LIBDIR)/Bio/AlignIO/nexus.pm $(INST_LIBDIR)/Bio/Annotation/Comment.pm $(INST_LIBDIR)/Bio/Factory/DriverFactory.pm $(INST_LIBDIR)/Bio/Tools/PRINTS/Parse.pm $(INST_LIBDIR)/Bio/DB/SeqI.pm $(INST_LIBDIR)/Bio/AlignIO/prodom.pm $(INST_LIBDIR)/Bio/Location/WidestCoordPolicy.pm $(INST_LIBDIR)/Bio/Root/IOManager.pm $(INST_LIBDIR)/Bio/LiveSeq/Repeat_Region.pm $(INST_LIBDIR)/Bio/Factory/SeqAnalysisParserFactoryI.pm $(INST_LIBDIR)/Bio/SeqIO/game/seqHandler.pm $(INST_LIBDIR)/Bio/LocationI.pm $(INST_LIBDIR)/Bio/Tools/Sim4/Results.pm $(INST_LIBDIR)/Bio/DB/Universal.pm $(INST_LIBDIR)/Bio/Tools/Prediction/Exon.pm $(INST_LIBDIR)/Bio/SeqIO/game/idHandler.pm $(INST_LIBDIR)/Bio/Search/Result/Fasta.pm $(INST_LIBDIR)/Bio/AlignIO/phylip.pm $(INST_LIBDIR)/Bio/Tools/Blast/HSP.pm $(INST_LIBDIR)/Bio/DB/WebDBSeqI.pm $(INST_LIBDIR)/Bio/SeqFeatureI.pm $(INST_LIBDIR)/Bio/SeqIO/swiss.pm $(INST_LIBDIR)/Bio/Search/Hit/Fasta.pm $(INST_LIBDIR)/Bio/Location/Split.pm $(INST_LIBDIR)/Bio/Tools/Grail.pm $(INST_LIBDIR)/Bio/DB/GenPept.pm $(INST_LIBDIR)/Bio/Tools/SeqStats.pm $(INST_LIBDIR)/Bio/LiveSeq/ChainI.pm $(INST_LIBDIR)/Bio/Variation/SeqDiff.pm $(INST_LIBDIR)/Bio/PrimarySeqI.pm $(INST_LIBDIR)/Bio/Variation/DNAMutation.pm $(INST_LIBDIR)/Bio/Search/Result/ResultI.pm $(INST_LIBDIR)/Bio/Search/Processor.pm $(INST_LIBDIR)/Bio/Tools/Prediction/Gene.pm $(INST_LIBDIR)/Bio/UnivAln.pm $(INST_LIBDIR)/Bio/Variation/Allele.pm $(INST_LIBDIR)/Bio/SeqFeature/FeaturePair.pm $(INST_LIBDIR)/biodesign.pod $(INST_LIBDIR)/Bio/AlignIO/msf.pm $(INST_LIBDIR)/Bio/SeqIO/embl.pm $(INST_LIBDIR)/Bio/LiveSeq/Repeat_Unit.pm $(INST_LIBDIR)/Bio/Tools/BPlite/Sbjct.pm $(INST_LIBDIR)/Bio/Tools/Fasta.pm $(INST_LIBDIR)/Bio/SeqFeature/Gene/Exon.pm $(INST_LIBDIR)/Bio/Root/IO.pm $(INST_LIBDIR)/Bio/Search/Hit/HitI.pm $(INST_LIBDIR)/Bio/DB/NCBIHelper.pm $(INST_LIBDIR)/Bio/Tools/Run/Alignment/TCoffee.pm $(INST_LIBDIR)/Bio/SeqIO/largefasta.pm $(INST_LIBDIR)/Bio/Index/Swissprot.pm $(INST_LIBDIR)/Bio/Tools/Run/Alignment/Clustalw.pm $(INST_LIBDIR)/Bio/Species.pm $(INST_LIBDIR)/Bio/Search/Processor/Fasta.pm $(INST_LIBDIR)/Bio/Index/Abstract.pm $(INST_LIBDIR)/Bio/Seq/LargePrimarySeq.pm $(INST_LIBDIR)/Bio/LiveSeq/Mutator.pm $(INST_LIBDIR)/Bio/AlignIO.pm $(INST_LIBDIR)/Bio/Tools/pSW.pm $(INST_LIBDIR)/Bio/Tools/BPlite/Iteration.pm $(INST_LIBDIR)/Bio/Factory/SeqAnalysisParserFactory.pm $(INST_LIBDIR)/Bio/SimpleAlign.pm $(INST_LIBDIR)/Bio/SeqFeature/UTR5.pm $(INST_LIBDIR)/Bio/Index/GenBank.pm $(INST_LIBDIR)/Bio/AlignIO/stockholm.pm $(INST_LIBDIR)/Bio/SeqIO.pm $(INST_LIBDIR)/Bio/Index/Fasta.pm $(INST_LIBDIR)/Bio/SearchDist.pm $(INST_LIBDIR)/Bio/LiveSeq/IO/BioPerl.pm $(INST_LIBDIR)/bptutorial.pl $(INST_LIBDIR)/Bio/SeqIO/game.pm $(INST_LIBDIR)/Bio/SeqIO/fasta.pm $(INST_LIBDIR)/Bio/Tools/AlignFactory.pm $(INST_LIBDIR)/Bio/SeqIO/game/featureHandler.pm $(INST_LIBDIR)/Bio/AlignIO/pfam.pm $(INST_LIBDIR)/Bio/SeqIO/raw.pm $(INST_LIBDIR)/Bio/Tools/BPbl2seq.pm $(INST_LIBDIR)/Bio/Tools/Blast/HTML.pm $(INST_LIBDIR)/Bio/Root/Utilities.pm $(INST_LIBDIR)/Bio/Tools/Blast.pm $(INST_LIBDIR)/Bio/Tools/PRFSCAN/Parse.pm $(INST_LIBDIR)/bioperl.pod $(INST_LIBDIR)/Bio/SeqFeature/Gene/GeneStructureI.pm $(INST_LIBDIR)/Bio/SeqIO/genbank.pm $(INST_LIBDIR)/Bio/AlignIO/bl2seq.pm $(INST_LIBDIR)/Bio/Tools/BPlite.pm $(INST_LIBDIR)/Bio/AlignIO/selex.pm $(INST_LIBDIR)/Bio/Range.pm $(INST_LIBDIR)/Bio/DB/SwissProt.pm $(INST_LIBDIR)/Bio/SeqFeature/Gene/TranscriptI.pm $(INST_LIBDIR)/Bio/DBLinkContainerI.pm $(INST_LIBDIR)/Bio/LiveSeq/Range.pm $(INST_LIBDIR)/Bio/SeqFeature/Gene/ExonI.pm $(INST_LIBDIR)/Bio/DB/Ace.pm $(INST_LIBDIR)/Bio/SeqFeature/Intron.pm $(INST_LIBDIR)/Bio/DB/RandomAccessI.pm $(INST_LIBDIR)/Bio/Index/SwissPfam.pm $(INST_LIBDIR)/Bio/Location/Fuzzy.pm $(INST_LIBDIR)/Bio/Tools/Blast/CHANGES $(INST_LIBDIR)/Bio/SeqIO/MultiFile.pm $(INST_LIBDIR)/Bio/Tools/RestrictionEnzyme.pm $(INST_LIBDIR)/Bio/Tools/HMMER/Results.pm $(INST_LIBDIR)/Bio/Location/SplitLocationI.pm $(INST_LIBDIR)/Bio/Variation/RNAChange.pm $(INST_LIBDIR)/Bio/LiveSeq/DNA.pm $(INST_LIBDIR)/Bio/Tools/Blast/README $(INST_LIBDIR)/Bio/Seq/LargeSeq.pm $(INST_LIBDIR)/Bio/Annotation.pm $(INST_LIBDIR)/Bio/LiveSeq/Exon.pm $(INST_LIBDIR)/Bio/Location/CoordinatePolicyI.pm $(INST_LIBDIR)/Bio/SeqFeature/Similarity.pm
	rm -rf Makefile Makefile.old ./Seq.pm.bak


# --- MakeMaker dist_basics section:

distclean :: realclean distcheck

distcheck :
	$(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -MExtUtils::Manifest=fullcheck \
		-e fullcheck

skipcheck :
	$(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -MExtUtils::Manifest=skipcheck \
		-e skipcheck

manifest :
	$(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -MExtUtils::Manifest=mkmanifest \
		-e mkmanifest


# --- MakeMaker dist_core section:

dist : $(DIST_DEFAULT)
	@$(PERL) -le 'print "Warning: Makefile possibly out of date with $$vf" if ' \
	    -e '-e ($$vf="$(VERSION_FROM)") and -M $$vf < -M "Makefile";'

tardist : $(DISTVNAME).tar$(SUFFIX)

zipdist : $(DISTVNAME).zip

$(DISTVNAME).tar$(SUFFIX) : distdir
	$(PREOP)
	$(TO_UNIX)
	$(TAR) $(TARFLAGS) $(DISTVNAME).tar $(DISTVNAME)
	$(RM_RF) $(DISTVNAME)
	$(COMPRESS) $(DISTVNAME).tar
	$(POSTOP)

$(DISTVNAME).zip : distdir
	$(PREOP)
	$(ZIP) $(ZIPFLAGS) $(DISTVNAME).zip $(DISTVNAME)
	$(RM_RF) $(DISTVNAME)
	$(POSTOP)

uutardist : $(DISTVNAME).tar$(SUFFIX)
	uuencode $(DISTVNAME).tar$(SUFFIX) \
		$(DISTVNAME).tar$(SUFFIX) > \
		$(DISTVNAME).tar$(SUFFIX)_uu

shdist : distdir
	$(PREOP)
	$(SHAR) $(DISTVNAME) > $(DISTVNAME).shar
	$(RM_RF) $(DISTVNAME)
	$(POSTOP)


# --- MakeMaker dist_dir section:

distdir :
	$(RM_RF) $(DISTVNAME)
	$(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -MExtUtils::Manifest=manicopy,maniread \
		-e "manicopy(maniread(),'$(DISTVNAME)', '$(DIST_CP)');"


# --- MakeMaker dist_test section:

disttest : distdir
	cd $(DISTVNAME) && $(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) Makefile.PL
	cd $(DISTVNAME) && $(MAKE)
	cd $(DISTVNAME) && $(MAKE) test


# --- MakeMaker dist_ci section:

ci :
	$(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -MExtUtils::Manifest=maniread \
		-e "@all = keys %{ maniread() };" \
		-e 'print("Executing $(CI) @all\n"); system("$(CI) @all");' \
		-e 'print("Executing $(RCS_LABEL) ...\n"); system("$(RCS_LABEL) @all");'


# --- MakeMaker install section:

install :: all pure_install doc_install

install_perl :: all pure_perl_install doc_perl_install

install_site :: all pure_site_install doc_site_install

install_ :: install_site
	@echo INSTALLDIRS not defined, defaulting to INSTALLDIRS=site

pure_install :: pure_$(INSTALLDIRS)_install

doc_install :: doc_$(INSTALLDIRS)_install
	@echo Appending installation info to $(INSTALLARCHLIB)/perllocal.pod

pure__install : pure_site_install
	@echo INSTALLDIRS not defined, defaulting to INSTALLDIRS=site

doc__install : doc_site_install
	@echo INSTALLDIRS not defined, defaulting to INSTALLDIRS=site

pure_perl_install ::
	@$(MOD_INSTALL) \
		read $(PERL_ARCHLIB)/auto/$(FULLEXT)/.packlist \
		write $(INSTALLARCHLIB)/auto/$(FULLEXT)/.packlist \
		$(INST_LIB) $(INSTALLPRIVLIB) \
		$(INST_ARCHLIB) $(INSTALLARCHLIB) \
		$(INST_BIN) $(INSTALLBIN) \
		$(INST_SCRIPT) $(INSTALLSCRIPT) \
		$(INST_HTMLLIBDIR) $(INSTALLHTMLPRIVLIBDIR) \
		$(INST_HTMLSCRIPTDIR) $(INSTALLHTMLSCRIPTDIR) \
		$(INST_MAN1DIR) $(INSTALLMAN1DIR) \
		$(INST_MAN3DIR) $(INSTALLMAN3DIR)
	@$(WARN_IF_OLD_PACKLIST) \
		$(SITEARCHEXP)/auto/$(FULLEXT)


pure_site_install ::
	@$(MOD_INSTALL) \
		read $(SITEARCHEXP)/auto/$(FULLEXT)/.packlist \
		write $(INSTALLSITEARCH)/auto/$(FULLEXT)/.packlist \
		$(INST_LIB) $(INSTALLSITELIB) \
		$(INST_ARCHLIB) $(INSTALLSITEARCH) \
		$(INST_BIN) $(INSTALLBIN) \
		$(INST_SCRIPT) $(INSTALLSCRIPT) \
		$(INST_HTMLLIBDIR) $(INSTALLHTMLSITELIBDIR) \
		$(INST_HTMLSCRIPTDIR) $(INSTALLHTMLSCRIPTDIR) \
		$(INST_MAN1DIR) $(INSTALLMAN1DIR) \
		$(INST_MAN3DIR) $(INSTALLMAN3DIR)
	@$(WARN_IF_OLD_PACKLIST) \
		$(PERL_ARCHLIB)/auto/$(FULLEXT)

doc_perl_install ::
	-@$(MKPATH) $(INSTALLARCHLIB)
	-@$(DOC_INSTALL) \
		"Module" "$(NAME)" \
		"installed into" "$(INSTALLPRIVLIB)" \
		LINKTYPE "$(LINKTYPE)" \
		VERSION "$(VERSION)" \
		EXE_FILES "$(EXE_FILES)" \
		>> $(INSTALLARCHLIB)/perllocal.pod

doc_site_install ::
	-@$(MKPATH) $(INSTALLARCHLIB)
	-@$(DOC_INSTALL) \
		"Module" "$(NAME)" \
		"installed into" "$(INSTALLSITELIB)" \
		LINKTYPE "$(LINKTYPE)" \
		VERSION "$(VERSION)" \
		EXE_FILES "$(EXE_FILES)" \
		>> $(INSTALLARCHLIB)/perllocal.pod


uninstall :: uninstall_from_$(INSTALLDIRS)dirs

uninstall_from_perldirs ::
	@$(UNINSTALL) $(PERL_ARCHLIB)/auto/$(FULLEXT)/.packlist

uninstall_from_sitedirs ::
	@$(UNINSTALL) $(SITEARCHEXP)/auto/$(FULLEXT)/.packlist


# --- MakeMaker force section:
# Phony target to force checking subdirectories.
FORCE:
	@$(NOOP)


# --- MakeMaker perldepend section:


# --- MakeMaker makefile section:

# We take a very conservative approach here, but it\'s worth it.
# We move Makefile to Makefile.old here to avoid gnu make looping.
Makefile : Makefile.PL $(CONFIGDEP)
	@echo "Makefile out-of-date with respect to $?"
	@echo "Cleaning current config before rebuilding Makefile..."
	-@$(RM_F) Makefile.old
	-@$(MV) Makefile Makefile.old
	-$(MAKE) -f Makefile.old clean $(DEV_NULL) || $(NOOP)
	$(PERL) "-I$(PERL_ARCHLIB)" "-I$(PERL_LIB)" Makefile.PL 
	@echo "==> Your Makefile has been rebuilt. <=="
	@echo "==> Please rerun the make command.  <=="
	false

# To change behavior to :: would be nice, but would break Tk b9.02
# so you find such a warning below the dist target.
#Makefile :: $(VERSION_FROM)
#	@echo "Warning: Makefile possibly out of date with $(VERSION_FROM)"


# --- MakeMaker staticmake section:

# --- MakeMaker makeaperl section ---
MAP_TARGET    = perl
FULLPERL      = /usr/bin/perl

$(MAP_TARGET) :: static $(MAKE_APERL_FILE)
	$(MAKE) -f $(MAKE_APERL_FILE) $@

$(MAKE_APERL_FILE) : $(FIRST_MAKEFILE)
	@echo Writing \"$(MAKE_APERL_FILE)\" for this $(MAP_TARGET)
	@$(PERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) \
		Makefile.PL DIR= \
		MAKEFILE=$(MAKE_APERL_FILE) LINKTYPE=static \
		MAKEAPERL=1 NORECURS=1 CCCDLFLAGS=


# --- MakeMaker test section:

TEST_VERBOSE=0
TEST_TYPE=test_$(LINKTYPE)
TEST_FILE = test.pl
TEST_FILES = t/*.t
TESTDB_SW = -d

testdb :: testdb_$(LINKTYPE)

test :: $(TEST_TYPE)

test_dynamic :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' $(TEST_FILES)

testdb_dynamic :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) $(TESTDB_SW) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) $(TEST_FILE)

test_ : test_dynamic

test_static :: test_dynamic
testdb_static :: testdb_dynamic


# --- MakeMaker ppd section:
# Creates a PPD (Perl Package Description) for a binary distribution.
ppd:
	@$(PERL) -e "print qq{<SOFTPKG NAME=\"bioperl\" VERSION=\"0,07,1pre,0\">\n}. qq{\t<TITLE>bioperl</TITLE>\n}. qq{\t<ABSTRACT></ABSTRACT>\n}. qq{\t<AUTHOR></AUTHOR>\n}. qq{\t<IMPLEMENTATION>\n}. qq{\t\t<OS NAME=\"$(OSNAME)\" />\n}. qq{\t\t<ARCHITECTURE NAME=\"i386-linux\" />\n}. qq{\t\t<CODEBASE HREF=\"\" />\n}. qq{\t</IMPLEMENTATION>\n}. qq{</SOFTPKG>\n}" > bioperl.ppd

# --- MakeMaker pm_to_blib section:

pm_to_blib: $(TO_INST_PM)
	@$(PERL) "-I$(INST_ARCHLIB)" "-I$(INST_LIB)" \
	"-I$(PERL_ARCHLIB)" "-I$(PERL_LIB)" -MExtUtils::Install \
        -e "pm_to_blib({qw{$(PM_TO_BLIB)}},'$(INST_LIB)/auto')"
	@$(TOUCH) $@


# --- MakeMaker selfdocument section:


# --- MakeMaker postamble section:
show_tests :
	@echo 'type make test_<subtest> to run'
	@echo 'AAChange \
AAReverseMutate \
AlignIO \
Allele \
Annotation \
BPbl2seq \
BPlite \
BPpsilite \
Blast \
Chain \
Clustalw \
CodonTable \
DB \
DBFasta \
DNAMutation \
EMBL_DB \
GDB \
GFF \
Genpred \
HSP \
HTML \
Index \
LiveSeq \
LocatableSeq \
Location \
Mutation \
Mutator \
OddCodes \
PrimarySeq \
RNAChange \
Range \
RangeI \
RemoteBlast \
RestrictionEnzyme \
RootI \
Sbjct \
SearchDist \
Seq \
SeqAnal \
SeqAnalysisParser \
SeqDiff \
SeqFeature \
SeqIO \
SeqPattern \
SeqStats \
SeqUtils \
Sigcleave \
Sim4 \
SimilarityPair \
SimpleAlign \
Species \
StandAloneBlast \
TCoffee \
Tempfile \
Tools \
UnivAln \
Variation_IO \
WWW \
exception \
game \
hmmer \
largefasta \
largepseq \
multiple_fasta \
'
test_AAChange :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/AAChange.t
test_AAReverseMutate :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/AAReverseMutate.t
test_AlignIO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/AlignIO.t
test_Allele :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Allele.t
test_Annotation :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Annotation.t
test_BPbl2seq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BPbl2seq.t
test_BPlite :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BPlite.t
test_BPpsilite :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/BPpsilite.t
test_Blast :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Blast.t
test_Chain :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Chain.t
test_Clustalw :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Clustalw.t
test_CodonTable :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/CodonTable.t
test_DB :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/DB.t
test_DBFasta :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/DBFasta.t
test_DNAMutation :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/DNAMutation.t
test_EMBL_DB :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/EMBL_DB.t
test_GDB :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/GDB.t
test_GFF :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/GFF.t
test_Genpred :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Genpred.t
test_HSP :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/HSP.t
test_HTML :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/HTML.t
test_Index :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Index.t
test_LiveSeq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/LiveSeq.t
test_LocatableSeq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/LocatableSeq.t
test_Location :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Location.t
test_Mutation :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Mutation.t
test_Mutator :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Mutator.t
test_OddCodes :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/OddCodes.t
test_PrimarySeq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/PrimarySeq.t
test_RNAChange :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RNAChange.t
test_Range :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Range.t
test_RangeI :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RangeI.t
test_RemoteBlast :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RemoteBlast.t
test_RestrictionEnzyme :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RestrictionEnzyme.t
test_RootI :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/RootI.t
test_Sbjct :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Sbjct.t
test_SearchDist :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SearchDist.t
test_Seq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Seq.t
test_SeqAnal :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqAnal.t
test_SeqAnalysisParser :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqAnalysisParser.t
test_SeqDiff :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqDiff.t
test_SeqFeature :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqFeature.t
test_SeqIO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqIO.t
test_SeqPattern :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqPattern.t
test_SeqStats :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqStats.t
test_SeqUtils :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SeqUtils.t
test_Sigcleave :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Sigcleave.t
test_Sim4 :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Sim4.t
test_SimilarityPair :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SimilarityPair.t
test_SimpleAlign :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/SimpleAlign.t
test_Species :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Species.t
test_StandAloneBlast :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/StandAloneBlast.t
test_TCoffee :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/TCoffee.t
test_Tempfile :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Tempfile.t
test_Tools :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Tools.t
test_UnivAln :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/UnivAln.t
test_Variation_IO :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/Variation_IO.t
test_WWW :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/WWW.t
test_exception :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/exception.t
test_game :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/game.t
test_hmmer :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/hmmer.t
test_largefasta :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/largefasta.t
test_largepseq :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/largepseq.t
test_multiple_fasta :: pure_all
	PERL_DL_NONLAZY=1 $(FULLPERL) -I$(INST_ARCHLIB) -I$(INST_LIB) -I$(PERL_ARCHLIB) -I$(PERL_LIB) -e 'use Test::Harness qw(&runtests $$verbose); $$verbose=$(TEST_VERBOSE); runtests @ARGV;' t/multiple_fasta.t



# End.
